<p>QIIME 2 (2017.10) is a widely used package to identity abundance of microbes using 16s ribosomal RNA (a “marker gene”). 16s rRNA has 9 hypervariable regions (V1-V9), which can be used to identify differen species.</p>

<p>Data produced by QIIME 2 exist as QIIME 2 artifacts. A QIIME 2 artifact typically has the .qza file extension when output data stored in a file. Visualizations are another type of data (.qzv file) generated by QIIME 2, which can be viewed using a web interface <a href="https://view.qiime2.org">https://view.qiime2.org</a> (at Firefox web browser) without requiring a QIIME installation. Since QIIME 2 works with artifacts instead of data files (e.g. FASTA files), we must create a QIIME 2 artifact by importing our fastq.gz data files.</p>

<p>Here are some bash scripts and qsub commands that submit QIIME jobs to the cluster. <code class="highlighter-rouge">qsub</code> is a job submission command to Sun Grid Engine (SGE) cluster.</p>

<p>To find the PHRED offset used for the positional quality scores, execute the following script (<a href="https://bitbucket.org/adinasarapu/clustercomputing/src/6e3396384fc31cdb4703534a4ca42d7f6a979954/fastq_manifest_phred.sh?at=master">BitBucket</a>).</p>
<div class="language-bash highlighter-rouge"><pre class="highlight"><code>sh fastq_manifest_phred.sh
</code></pre>
</div>
<div class="language-bash highlighter-rouge"><pre class="highlight"><code><span class="c">#!/bin/sh</span>
<span class="c"># fastq_manifest_phred.sh</span>
<span class="c"># checks-fastq-quality-score-format</span>
<span class="nv">FASTQ_FILE</span><span class="o">=</span><span class="nv">$HOME</span>/path/to/fastq.gz
zcat <span class="k">${</span><span class="nv">FASTQ_FILE</span><span class="k">}</span> <span class="se">\</span>
	| awk <span class="s1">'NR%4==0 {printf $0}'</span> <span class="se">\</span>
	| tr -d <span class="s1">'\n'</span> <span class="se">\</span>
	| hexdump -v -e<span class="s1">'/1 "%u\n"'</span> <span class="se">\</span>
	| head -n100000 <span class="se">\</span>
	| sort -u
</code></pre>
</div>
<p>To import FASTQ files as QIIME 2 artifact and plot sequence positional quality scores, download the <a href="https://bitbucket.org/adinasarapu/clustercomputing/src/bcc4d689076a1d977da67b4f411612f2addf3920/qiime2.2017.10_step1.sh?at=master&amp;fileviewer=file-view-default">qiime2.2017.10_step1.sh</a> from BitBucket and execute as</p>
<div class="language-bash highlighter-rouge"><pre class="highlight"><code>qsub qiime2.2017.10_step1.sh
</code></pre>
</div>
<p>To denoise imported paired sequence data and filter chimeras, QIIME 2 uses DADA2, which is an open-source software package that denoises and removes sequencing errors from Illumina amplicon sequence data, download the <a href="https://bitbucket.org/adinasarapu/clustercomputing/src/bcc4d689076a1d977da67b4f411612f2addf3920/qiime2.2017.10_step2.sh?at=master&amp;fileviewer=file-view-default">qiime2.2017.10_step2.sh</a> from BitBucket and execute as</p>
<div class="language-bash highlighter-rouge"><pre class="highlight"><code>qsub qiime2.2017.10_step2.sh
</code></pre>
</div>
<p>For feature filtering (i.e., removal of samples and features from a feature table), download the <a href="https://bitbucket.org/adinasarapu/clustercomputing/src/bcc4d689076a1d977da67b4f411612f2addf3920/qiime2.2017.10_step3.sh?at=master&amp;fileviewer=file-view-default">qiime2.2017.10_step3.sh</a> from BitBucket and execute as</p>
<div class="language-bash highlighter-rouge"><pre class="highlight"><code>qsub qiime2.2017.10_step3.sh
</code></pre>
</div>
<p>For diversity (alpha and beta), taxonomic and comratative analyses, download the <a href="https://bitbucket.org/adinasarapu/clustercomputing/src/bcc4d689076a1d977da67b4f411612f2addf3920/qiime2.2017.10_step4.sh?at=master&amp;fileviewer=file-view-default">qiime2.2017.10_step4.sh</a> from BitBucket and execute as</p>
<div class="language-bash highlighter-rouge"><pre class="highlight"><code>qsub qiime2.2017.10_step4.sh
</code></pre>
</div>
<p><img src="/images/PC1vsPC2plot_100px.png" alt="PC1vsPC2plot" /></p>

<p>The above analyses also produce a key summary table or BIOM (Biological Observation Matrix) file containing feature (Operational Taxonomic Units, OTUs) abundance information across samples, along with various annotations and sample metadata. Alternatively, upload BIOM file to <a href="http://www.microbiomeanalyst.ca">MicrobiomeAnalyst</a>, a web-based tool for comprehensive statistical, visual and meta-analysis of microbiome data, for <b>taxonomic profiling</b> - to characterize community compositions based on methods developed in ecology such as alpha-diversity (within-sample diversity) or beta-diversity (between-sample diversity) and <b>comparative analysis</b> - to identify features that are significantly different among conditions under study.</p>

<p>After analyzing your data, it’s finally time to interpret your results!</p>
