

<!doctype html>
<html lang="en" class="no-js">
  <head>
    

<meta charset="utf-8">



<!-- begin SEO -->









<title>Posts by Tags - Ashok R. Dinasarapu</title>







<meta property="og:locale" content="en-US">
<meta property="og:site_name" content="Ashok R. Dinasarapu">
<meta property="og:title" content="Posts by Tags">


  <link rel="canonical" href="http://localhost:4000/tags/">
  <meta property="og:url" content="http://localhost:4000/tags/">







  

  












  <script type="application/ld+json">
    {
      "@context" : "http://schema.org",
      "@type" : "Person",
      "name" : "Ashok R. Dinasarapu",
      "url" : "http://localhost:4000",
      "sameAs" : null
    }
  </script>






<!-- end SEO -->


<link href="http://localhost:4000/feed.xml" type="application/atom+xml" rel="alternate" title="Ashok R. Dinasarapu Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="http://localhost:4000/assets/css/main.css">

<meta http-equiv="cleartype" content="on">
    

<!-- start custom head snippets -->

<link rel="apple-touch-icon" sizes="57x57" href="http://localhost:4000/images/apple-touch-icon-57x57.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="60x60" href="http://localhost:4000/images/apple-touch-icon-60x60.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="72x72" href="http://localhost:4000/images/apple-touch-icon-72x72.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="76x76" href="http://localhost:4000/images/apple-touch-icon-76x76.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="114x114" href="http://localhost:4000/images/apple-touch-icon-114x114.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="120x120" href="http://localhost:4000/images/apple-touch-icon-120x120.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="144x144" href="http://localhost:4000/images/apple-touch-icon-144x144.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="152x152" href="http://localhost:4000/images/apple-touch-icon-152x152.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="180x180" href="http://localhost:4000/images/apple-touch-icon-180x180.png?v=M44lzPylqQ">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-32x32.png?v=M44lzPylqQ" sizes="32x32">
<link rel="icon" type="image/png" href="http://localhost:4000/images/android-chrome-192x192.png?v=M44lzPylqQ" sizes="192x192">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-96x96.png?v=M44lzPylqQ" sizes="96x96">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-16x16.png?v=M44lzPylqQ" sizes="16x16">
<link rel="manifest" href="http://localhost:4000/images/manifest.json?v=M44lzPylqQ">
<link rel="mask-icon" href="http://localhost:4000/images/safari-pinned-tab.svg?v=M44lzPylqQ" color="#000000">
<link rel="shortcut icon" href="/images/favicon.ico?v=M44lzPylqQ">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-TileImage" content="http://localhost:4000/images/mstile-144x144.png?v=M44lzPylqQ">
<meta name="msapplication-config" content="http://localhost:4000/images/browserconfig.xml?v=M44lzPylqQ">
<meta name="theme-color" content="#ffffff">
<link rel="stylesheet" href="http://localhost:4000/assets/css/academicons.css"/>
<!-- end custom head snippets -->

  </head>

  <body>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->
    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <button><div class="navicon"></div></button>
        <ul class="visible-links">
          <li class="masthead__menu-item masthead__menu-item--lg"><a href="http://localhost:4000/">Ashok R. Dinasarapu</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/publications/">Publications</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/teaching/">Teaching</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/year-archive/">NGS & BigData</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/cv/">Résumé</a></li>
          
        </ul>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>

    



<div id="main" role="main">
  


  <div class="sidebar sticky">
  



<div itemscope itemtype="http://schema.org/Person">

  <div class="author__avatar">
    
    	<img src="http://localhost:4000/images/profile.png" class="author__avatar" alt="Scientist, Bioinformatics">
    
  </div>

  <div class="author__content">
    <h3 class="author__name">Scientist, Bioinformatics</h3>
    <p class="author__bio">Emory University</p>
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      
        <li><i class="fa fa-fw fa-map-marker" aria-hidden="true"></i> Atlanta, GA</li>
      
      
      
      
        <li><a href="mailto:ashok.reddy.dinasarapu@emory.edu"><i class="fa fa-fw fa-envelope-square" aria-hidden="true"></i> Email</a></li>
      
      
      
        <li><a href="https://twitter.com/adinasarapu"><i class="fa fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
      
      
      
      
        <li><a href="https://www.linkedin.com/in/dareddy"><i class="fa fa-fw fa-linkedin-square" aria-hidden="true"></i> LinkedIn</a></li>
      
      
      
      
      
        <li><a href="https://bitbucket.org/adinasarapu"><i class="fa fa-fw fa-bitbucket" aria-hidden="true"></i> Bitbucket</a></li>
      
      
        <li><a href="https://github.com/adinasarapu"><i class="fa fa-fw fa-github" aria-hidden="true"></i> Github</a></li>
      
      
      
      
      
      
      
      
      
      
      
      
      
      
        <li><a href="https://scholar.google.com/citations?user=b6GBykAAAAAJ"><i class="ai ai-google-scholar-square ai-fw"></i> Google Scholar</a></li>
      
      
        <li><a href="http://orcid.org/0000-0002-1423-1518"><i class="ai ai-orcid-square ai-fw"></i> ORCID</a></li>
      
      
    </ul>
  </div>
</div>

  
  </div>


  <div class="archive">
    
      <h1 class="page__title">Posts by Tags</h1>
    
    
<!--
# Jekyll Group-By-Array 0.1.0
# https://github.com/mushishi78/jekyll-group-by-array
# © 2015 Max White <mushishi78@gmail.com>
# MIT License
-->

<!-- Initialize -->




<!-- Map -->


<!-- Flatten -->


<!-- Uniq -->



<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->







<!-- group_items -->


<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->




  
  <h2 id="10xgenomics" class="archive__subtitle">10XGenomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-08:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">cellranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="10plex" class="archive__subtitle">10plex</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-09T00:00:00-08:00">January 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="16s-rrna-amplicon" class="archive__subtitle">16s rRNA amplicon</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-08:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="atac-seq-assay-for-transposase-accessible-chromatin-with-high-throughput-sequencing" class="archive__subtitle">ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing)</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/12/blog-post-atacseq/" rel="permalink">ATAC-seq peak calling with MACS2
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-12-05T00:00:00-08:00">December 05, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing) is a next-generation sequencing approach for the analysis of open chromatin regions to assess the genome-wise chromatin accessibility.

<strong><a href="http://localhost:4000/posts/2019/12/blog-post-atacseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="bwa" class="archive__subtitle">BWA</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/12/blog-post-atacseq/" rel="permalink">ATAC-seq peak calling with MACS2
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-12-05T00:00:00-08:00">December 05, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing) is a next-generation sequencing approach for the analysis of open chromatin regions to assess the genome-wise chromatin accessibility.

<strong><a href="http://localhost:4000/posts/2019/12/blog-post-atacseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="cell-ranger" class="archive__subtitle">Cell Ranger</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-08:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">cellranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="chromium-single-cell-gene-expression" class="archive__subtitle">Chromium Single Cell Gene Expression</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-08:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">cellranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="cluster-computing" class="archive__subtitle">Cluster Computing</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-08:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">cellranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="dada2" class="archive__subtitle">DADA2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-08:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="fastqc" class="archive__subtitle">FastQC</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-07:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).

<strong><a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="feature-barcoding" class="archive__subtitle">Feature Barcoding</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-08:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">cellranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="gatk" class="archive__subtitle">GATK</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-08:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.

<strong><a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="homer" class="archive__subtitle">HOMER</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/12/blog-post-atacseq/" rel="permalink">ATAC-seq peak calling with MACS2
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-12-05T00:00:00-08:00">December 05, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing) is a next-generation sequencing approach for the analysis of open chromatin regions to assess the genome-wise chromatin accessibility.

<strong><a href="http://localhost:4000/posts/2019/12/blog-post-atacseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="htseq" class="archive__subtitle">HTSeq</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-07:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).

<strong><a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="humann2" class="archive__subtitle">HUMAnN2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-07:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.

<strong><a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="hadoop-distributed-file-system" class="archive__subtitle">Hadoop Distributed File System</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink">Building a real-time big data pipeline (part 3: Spark, SQL)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  9 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-15T00:00:00-08:00">February 15, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is an open-source cluster computing system that provides high-level API in Java, Scala, Python and R.<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup> Spark also packaged with higher-level libraries for SQL, machine learning, streaming, and graphs.

Spark SQL is Spark’s package for working with structured data.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink">Building a real-time big data pipeline (part 2: Spark, Hadoop)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  8 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-09T00:00:00-08:00">February 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is a general-purpose, in-memory cluster computing engine  for large scale data processing<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>. Apache Spark provides APIs in Java, Scala, Python, and R.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="indel" class="archive__subtitle">Indel</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-08:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.

<strong><a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="lfq" class="archive__subtitle">LFQ</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-04-07T00:00:00-07:00">April 07, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Liquid chromatography (LC) coupled with mass spectrometry (MS) has been widely used for protein expression quantification. Protein quantification by tandem-MS (MS/MS) uses integrated peak intensity from the parent-ion mass (MS1) or features from fragment-ions (MS2). MS1 methods use the iBAQ (intensity Based Absolute Quantification) algorithm (a protein’s total non-normalised intensities are divided by the number of measurable tryptic peptides). Untargeted label-free quantitation (LFQ) of proteins, aims to determine the relative amount of proteins in two or more biological samples.

<strong><a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="macs2" class="archive__subtitle">MACS2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/12/blog-post-atacseq/" rel="permalink">ATAC-seq peak calling with MACS2
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-12-05T00:00:00-08:00">December 05, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing) is a next-generation sequencing approach for the analysis of open chromatin regions to assess the genome-wise chromatin accessibility.

<strong><a href="http://localhost:4000/posts/2019/12/blog-post-atacseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="ms3-spectra" class="archive__subtitle">MS3 spectra</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-09T00:00:00-08:00">January 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="matrixeqtl" class="archive__subtitle">MatrixEqtl</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-08:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="maxquant" class="archive__subtitle">MaxQuant</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-09T00:00:00-08:00">January 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-04-07T00:00:00-07:00">April 07, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Liquid chromatography (LC) coupled with mass spectrometry (MS) has been widely used for protein expression quantification. Protein quantification by tandem-MS (MS/MS) uses integrated peak intensity from the parent-ion mass (MS1) or features from fragment-ions (MS2). MS1 methods use the iBAQ (intensity Based Absolute Quantification) algorithm (a protein’s total non-normalised intensities are divided by the number of measurable tryptic peptides). Untargeted label-free quantitation (LFQ) of proteins, aims to determine the relative amount of proteins in two or more biological samples.

<strong><a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="metaphlan2" class="archive__subtitle">MetaPhlAn2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-07:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.

<strong><a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="mysql" class="archive__subtitle">MySQL</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink">Building a real-time big data pipeline (part 3: Spark, SQL)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  9 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-15T00:00:00-08:00">February 15, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is an open-source cluster computing system that provides high-level API in Java, Scala, Python and R.<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup> Spark also packaged with higher-level libraries for SQL, machine learning, streaming, and graphs.

Spark SQL is Spark’s package for working with structured data.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="otu" class="archive__subtitle">OTU</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-08:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="ptxqc" class="archive__subtitle">PTXQC</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-04-07T00:00:00-07:00">April 07, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Liquid chromatography (LC) coupled with mass spectrometry (MS) has been widely used for protein expression quantification. Protein quantification by tandem-MS (MS/MS) uses integrated peak intensity from the parent-ion mass (MS1) or features from fragment-ions (MS2). MS1 methods use the iBAQ (intensity Based Absolute Quantification) algorithm (a protein’s total non-normalised intensities are divided by the number of measurable tryptic peptides). Untargeted label-free quantitation (LFQ) of proteins, aims to determine the relative amount of proteins in two or more biological samples.

<strong><a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="perseus" class="archive__subtitle">Perseus</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-09T00:00:00-08:00">January 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-04-07T00:00:00-07:00">April 07, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Liquid chromatography (LC) coupled with mass spectrometry (MS) has been widely used for protein expression quantification. Protein quantification by tandem-MS (MS/MS) uses integrated peak intensity from the parent-ion mass (MS1) or features from fragment-ions (MS2). MS1 methods use the iBAQ (intensity Based Absolute Quantification) algorithm (a protein’s total non-normalised intensities are divided by the number of measurable tryptic peptides). Untargeted label-free quantitation (LFQ) of proteins, aims to determine the relative amount of proteins in two or more biological samples.

<strong><a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="proteomics" class="archive__subtitle">Proteomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-09T00:00:00-08:00">January 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-04-07T00:00:00-07:00">April 07, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Liquid chromatography (LC) coupled with mass spectrometry (MS) has been widely used for protein expression quantification. Protein quantification by tandem-MS (MS/MS) uses integrated peak intensity from the parent-ion mass (MS1) or features from fragment-ions (MS2). MS1 methods use the iBAQ (intensity Based Absolute Quantification) algorithm (a protein’s total non-normalised intensities are divided by the number of measurable tryptic peptides). Untargeted label-free quantitation (LFQ) of proteins, aims to determine the relative amount of proteins in two or more biological samples.

<strong><a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="rna-seq" class="archive__subtitle">RNA-Seq</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-07:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).

<strong><a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-08:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.

<strong><a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-08:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="rnaseq" class="archive__subtitle">RNASeq</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-07:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).

<strong><a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="rawmeat" class="archive__subtitle">RawMeat</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-04-07T00:00:00-07:00">April 07, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Liquid chromatography (LC) coupled with mass spectrometry (MS) has been widely used for protein expression quantification. Protein quantification by tandem-MS (MS/MS) uses integrated peak intensity from the parent-ion mass (MS1) or features from fragment-ions (MS2). MS1 methods use the iBAQ (intensity Based Absolute Quantification) algorithm (a protein’s total non-normalised intensities are divided by the number of measurable tryptic peptides). Untargeted label-free quantitation (LFQ) of proteins, aims to determine the relative amount of proteins in two or more biological samples.

<strong><a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="snp" class="archive__subtitle">SNP</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-08:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.

<strong><a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="star" class="archive__subtitle">STAR</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-07:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).

<strong><a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-08:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.

<strong><a href="http://localhost:4000/posts/2018/01/blog-post-gatk/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="seurat" class="archive__subtitle">Seurat</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-08:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">cellranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="single-cell" class="archive__subtitle">Single Cell</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-08:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">cellranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="single-cell-rna-sequencing-scrna-seq" class="archive__subtitle">Single cell RNA-sequencing (scRNA-seq)</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-08:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">cellranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="spring-boot" class="archive__subtitle">Spring Boot</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink">Building a real-time big data pipeline (part 1: Kafka)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-26T00:00:00-08:00">January 26, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><a href="https://kafka.apache.org">Kafka</a> is used for building real-time data pipelines and streaming apps.

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="tmt" class="archive__subtitle">TMT</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-09T00:00:00-08:00">January 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="tmt-10plex" class="archive__subtitle">TMT 10plex</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-09T00:00:00-08:00">January 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="yaml" class="archive__subtitle">YAML</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink">Building a real-time big data pipeline (part 1: Kafka)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-26T00:00:00-08:00">January 26, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><a href="https://kafka.apache.org">Kafka</a> is used for building real-time data pipelines and streaming apps.

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="yarn" class="archive__subtitle">YARN</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink">Building a real-time big data pipeline (part 2: Spark, Hadoop)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  8 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-09T00:00:00-08:00">February 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is a general-purpose, in-memory cluster computing engine  for large scale data processing<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>. Apache Spark provides APIs in Java, Scala, Python, and R.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="zookeeper" class="archive__subtitle">Zookeeper</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink">Building a real-time big data pipeline (part 1: Kafka)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-26T00:00:00-08:00">January 26, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><a href="https://kafka.apache.org">Kafka</a> is used for building real-time data pipelines and streaming apps.

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="apache-kafka" class="archive__subtitle">apache kafka</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink">Building a real-time big data pipeline (part 1: Kafka)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-26T00:00:00-08:00">January 26, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><a href="https://kafka.apache.org">Kafka</a> is used for building real-time data pipelines and streaming apps.

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="apache-spark" class="archive__subtitle">apache spark</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink">Building a real-time big data pipeline (part 3: Spark, SQL)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  9 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-15T00:00:00-08:00">February 15, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is an open-source cluster computing system that provides high-level API in Java, Scala, Python and R.<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup> Spark also packaged with higher-level libraries for SQL, machine learning, streaming, and graphs.

Spark SQL is Spark’s package for working with structured data.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink">Building a real-time big data pipeline (part 2: Spark, Hadoop)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  8 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-09T00:00:00-08:00">February 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is a general-purpose, in-memory cluster computing engine  for large scale data processing<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>. Apache Spark provides APIs in Java, Scala, Python, and R.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="big-data" class="archive__subtitle">big data</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink">Building a real-time big data pipeline (part 3: Spark, SQL)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  9 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-15T00:00:00-08:00">February 15, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is an open-source cluster computing system that provides high-level API in Java, Scala, Python and R.<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup> Spark also packaged with higher-level libraries for SQL, machine learning, streaming, and graphs.

Spark SQL is Spark’s package for working with structured data.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink">Building a real-time big data pipeline (part 2: Spark, Hadoop)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  8 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-09T00:00:00-08:00">February 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is a general-purpose, in-memory cluster computing engine  for large scale data processing<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>. Apache Spark provides APIs in Java, Scala, Python, and R.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink">Building a real-time big data pipeline (part 1: Kafka)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-26T00:00:00-08:00">January 26, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><a href="https://kafka.apache.org">Kafka</a> is used for building real-time data pipelines and streaming apps.

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="bioinformatics" class="archive__subtitle">bioinformatics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink">Building a real-time big data pipeline (part 3: Spark, SQL)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  9 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-15T00:00:00-08:00">February 15, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is an open-source cluster computing system that provides high-level API in Java, Scala, Python and R.<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup> Spark also packaged with higher-level libraries for SQL, machine learning, streaming, and graphs.

Spark SQL is Spark’s package for working with structured data.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink">Building a real-time big data pipeline (part 2: Spark, Hadoop)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  8 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-09T00:00:00-08:00">February 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is a general-purpose, in-memory cluster computing engine  for large scale data processing<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>. Apache Spark provides APIs in Java, Scala, Python, and R.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="cnv" class="archive__subtitle">cnv</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/09/blog-post-trio/" rel="permalink">Copy number variation discovery workflows using NGS  data.
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-19T00:00:00-07:00">August 19, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Copy number variations (CNVs) represent gain or loss of genomic regions. CNVs transmit from parents to offspring or arise <i>de novo</i> and play important role in neuro-psychiatric disorders and cancers.

<strong><a href="http://localhost:4000/posts/2017/09/blog-post-trio/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="computing" class="archive__subtitle">computing</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-08:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/09/blog-post-trio/" rel="permalink">Copy number variation discovery workflows using NGS  data.
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-19T00:00:00-07:00">August 19, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Copy number variations (CNVs) represent gain or loss of genomic regions. CNVs transmit from parents to offspring or arise <i>de novo</i> and play important role in neuro-psychiatric disorders and cancers.

<strong><a href="http://localhost:4000/posts/2017/09/blog-post-trio/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="curation" class="archive__subtitle">curation</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publication/2013-ucsd-sgmp" rel="permalink">A comprehensive map of the human complement system
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-20T00:00:00-07:00">August 20, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The human complement activation takes place through one or more of the well-established pathways consisting of plasma and membrane-bound proteins.
<strong><a href="http://localhost:4000/publication/2013-ucsd-sgmp" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="discordant-sex-informaion" class="archive__subtitle">discordant sex informaion</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-08:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="docker" class="archive__subtitle">docker</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink">Building a real-time big data pipeline (part 1: Kafka)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-26T00:00:00-08:00">January 26, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><a href="https://kafka.apache.org">Kafka</a> is used for building real-time data pipelines and streaming apps.

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="eqtl" class="archive__subtitle">eQTL</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-08:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="edger" class="archive__subtitle">edgeR</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-07:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).

<strong><a href="http://localhost:4000/posts/2018/03/blog-post-rnaseq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-08:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="functional-profiling" class="archive__subtitle">functional profiling</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-07:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.

<strong><a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="genomics" class="archive__subtitle">genomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/09/blog-post-trio/" rel="permalink">Copy number variation discovery workflows using NGS  data.
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-19T00:00:00-07:00">August 19, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Copy number variations (CNVs) represent gain or loss of genomic regions. CNVs transmit from parents to offspring or arise <i>de novo</i> and play important role in neuro-psychiatric disorders and cancers.

<strong><a href="http://localhost:4000/posts/2017/09/blog-post-trio/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="hadoop" class="archive__subtitle">hadoop</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink">Building a real-time big data pipeline (part 3: Spark, SQL)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  9 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-15T00:00:00-08:00">February 15, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is an open-source cluster computing system that provides high-level API in Java, Scala, Python and R.<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup> Spark also packaged with higher-level libraries for SQL, machine learning, streaming, and graphs.

Spark SQL is Spark’s package for working with structured data.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink">Building a real-time big data pipeline (part 2: Spark, Hadoop)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  8 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-09T00:00:00-08:00">February 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is a general-purpose, in-memory cluster computing engine  for large scale data processing<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>. Apache Spark provides APIs in Java, Scala, Python, and R.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="heterozygosity-rate" class="archive__subtitle">heterozygosity rate</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-08:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="htseq" class="archive__subtitle">htseq</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-08:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-eqtl/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="human-complement" class="archive__subtitle">human complement</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publication/2013-ucsd-sgmp" rel="permalink">A comprehensive map of the human complement system
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-20T00:00:00-07:00">August 20, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The human complement activation takes place through one or more of the well-established pathways consisting of plasma and membrane-bound proteins.
<strong><a href="http://localhost:4000/publication/2013-ucsd-sgmp" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="ibac" class="archive__subtitle">iBAC</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-04-07T00:00:00-07:00">April 07, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Liquid chromatography (LC) coupled with mass spectrometry (MS) has been widely used for protein expression quantification. Protein quantification by tandem-MS (MS/MS) uses integrated peak intensity from the parent-ion mass (MS1) or features from fragment-ions (MS2). MS1 methods use the iBAQ (intensity Based Absolute Quantification) algorithm (a protein’s total non-normalised intensities are divided by the number of measurable tryptic peptides). Untargeted label-free quantitation (LFQ) of proteins, aims to determine the relative amount of proteins in two or more biological samples.

<strong><a href="http://localhost:4000/posts/2018/04/blog-post-lfq/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="isobaric-labeling" class="archive__subtitle">isobaric labeling</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-09T00:00:00-08:00">January 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="java" class="archive__subtitle">java</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink">Building a real-time big data pipeline (part 1: Kafka)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-26T00:00:00-08:00">January 26, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><a href="https://kafka.apache.org">Kafka</a> is used for building real-time data pipelines and streaming apps.

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="metagenomics" class="archive__subtitle">metagenomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-07:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.

<strong><a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="microbiome" class="archive__subtitle">microbiome</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-08:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="missingness" class="archive__subtitle">missingness</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-08:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="networks" class="archive__subtitle">networks</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publication/2013-ucsd-sgmp" rel="permalink">A comprehensive map of the human complement system
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-20T00:00:00-07:00">August 20, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The human complement activation takes place through one or more of the well-established pathways consisting of plasma and membrane-bound proteins.
<strong><a href="http://localhost:4000/publication/2013-ucsd-sgmp" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="plink" class="archive__subtitle">plink</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-08:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="qiime2" class="archive__subtitle">qiime2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-08:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.

<strong><a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="real-time-data-pipelines" class="archive__subtitle">real time data pipelines</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink">Building a real-time big data pipeline (part 3: Spark, SQL)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  9 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-15T00:00:00-08:00">February 15, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is an open-source cluster computing system that provides high-level API in Java, Scala, Python and R.<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup> Spark also packaged with higher-level libraries for SQL, machine learning, streaming, and graphs.

Spark SQL is Spark’s package for working with structured data.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink">Building a real-time big data pipeline (part 2: Spark, Hadoop)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  8 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-09T00:00:00-08:00">February 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is a general-purpose, in-memory cluster computing engine  for large scale data processing<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>. Apache Spark provides APIs in Java, Scala, Python, and R.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink">Building a real-time big data pipeline (part 1: Kafka)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-26T00:00:00-08:00">January 26, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><a href="https://kafka.apache.org">Kafka</a> is used for building real-time data pipelines and streaming apps.

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-kafka/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="runs-of-homozygosity" class="archive__subtitle">runs of homozygosity</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-08:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.

<strong><a href="http://localhost:4000/posts/2017/12/blog-post-plink/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="scala" class="archive__subtitle">scala</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink">Building a real-time big data pipeline (part 3: Spark, SQL)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  9 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-15T00:00:00-08:00">February 15, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is an open-source cluster computing system that provides high-level API in Java, Scala, Python and R.<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup> Spark also packaged with higher-level libraries for SQL, machine learning, streaming, and graphs.

Spark SQL is Spark’s package for working with structured data.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark-sql/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink">Building a real-time big data pipeline (part 2: Spark, Hadoop)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  8 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-02-09T00:00:00-08:00">February 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Apache Spark is a general-purpose, in-memory cluster computing engine  for large scale data processing<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>. Apache Spark provides APIs in Java, Scala, Python, and R.

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <a href="https://spark.apache.org">Apache Spark</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>
    </li>
  </ol>
</div>
<strong><a href="http://localhost:4000/posts/2020/02/blog-post-spark/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="shotgun-metagenomics-sequencing" class="archive__subtitle">shotgun metagenomics sequencing</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-07:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.

<strong><a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="tandem-mass-tag" class="archive__subtitle">tandem mass tag</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-01-09T00:00:00-08:00">January 09, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).

<strong><a href="http://localhost:4000/posts/2020/01/blog-post-tmt/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  
  <h2 id="taxonomic-profiling" class="archive__subtitle">taxonomic profiling</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-07:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.

<strong><a href="http://localhost:4000/posts/2018/07/blog-post-metagenomics/" rel="permalink"> Read more</a></strong></p></p>
    
    
    

  </article>
</div>

  

  </div>
</div>


    </script>

    <div class="page__footer">
      <footer>
        <!-- start custom footer snippets -->
<!--<a href="/sitemap/">Sitemap</a> -->
<!-- end custom footer snippets -->

        

<!--
<div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    
    
    
    
      <li><a href="http://github.com/adinasarapu"><i class="fa fa-fw fa-github" aria-hidden="true"></i> GitHub</a></li>
    
    
      <li><a href="http://bitbucket.org/adinasarapu"><i class="fa fa-fw fa-bitbucket" aria-hidden="true"></i> Bitbucket</a></li>
    
    <li><a href="http://localhost:4000/feed.xml"><i class="fa fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
  </ul>
</div>
-->

<div class="page__footer-copyright">&copy; 2020 Ashok R. Dinasarapu  | <a href="http://adinasarapu.github.io/terms/" rel="nofollow">Privacy Policy</a>

<!-- Powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://github.com/academicpages/academicpages.github.io">AcademicPages</a>, a fork of <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.--> 
</div>

      </footer>
    </div>

    <script src="http://localhost:4000/assets/js/main.min.js"></script>




  <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-68614074-1', 'auto');
  ga('send', 'pageview');
</script>






  </body>
</html>

