

<!doctype html>
<html lang="en" class="no-js">
  <head>
    

<meta charset="utf-8">



<!-- begin SEO -->









<title>Posts by Tags - Ashok R. Dinasarapu Ph.D</title>







<meta property="og:locale" content="en-US">
<meta property="og:site_name" content="Ashok R. Dinasarapu Ph.D">
<meta property="og:title" content="Posts by Tags">


  <link rel="canonical" href="https://adinasarapu.github.io/tags/">
  <meta property="og:url" content="https://adinasarapu.github.io/tags/">







  

  












  <script type="application/ld+json">
    {
      "@context" : "http://schema.org",
      "@type" : "Person",
      "name" : "Ashok R. Dinasarapu",
      "url" : "https://adinasarapu.github.io",
      "sameAs" : null
    }
  </script>






<!-- end SEO -->


<link href="https://adinasarapu.github.io/feed.xml" type="application/atom+xml" rel="alternate" title="Ashok R. Dinasarapu Ph.D Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="https://adinasarapu.github.io/assets/css/main.css">

<meta http-equiv="cleartype" content="on">
    

<!-- start custom head snippets -->

<link rel="apple-touch-icon" sizes="57x57" href="https://adinasarapu.github.io/images/apple-touch-icon-57x57.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="60x60" href="https://adinasarapu.github.io/images/apple-touch-icon-60x60.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="72x72" href="https://adinasarapu.github.io/images/apple-touch-icon-72x72.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="76x76" href="https://adinasarapu.github.io/images/apple-touch-icon-76x76.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="114x114" href="https://adinasarapu.github.io/images/apple-touch-icon-114x114.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="120x120" href="https://adinasarapu.github.io/images/apple-touch-icon-120x120.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="144x144" href="https://adinasarapu.github.io/images/apple-touch-icon-144x144.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="152x152" href="https://adinasarapu.github.io/images/apple-touch-icon-152x152.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="180x180" href="https://adinasarapu.github.io/images/apple-touch-icon-180x180.png?v=M44lzPylqQ">
<link rel="icon" type="image/png" href="https://adinasarapu.github.io/images/favicon-32x32.png?v=M44lzPylqQ" sizes="32x32">
<link rel="icon" type="image/png" href="https://adinasarapu.github.io/images/android-chrome-192x192.png?v=M44lzPylqQ" sizes="192x192">
<link rel="icon" type="image/png" href="https://adinasarapu.github.io/images/favicon-96x96.png?v=M44lzPylqQ" sizes="96x96">
<link rel="icon" type="image/png" href="https://adinasarapu.github.io/images/favicon-16x16.png?v=M44lzPylqQ" sizes="16x16">
<link rel="manifest" href="https://adinasarapu.github.io/images/manifest.json?v=M44lzPylqQ">
<link rel="mask-icon" href="https://adinasarapu.github.io/images/safari-pinned-tab.svg?v=M44lzPylqQ" color="#000000">
<link rel="shortcut icon" href="/images/favicon.ico?v=M44lzPylqQ">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-TileImage" content="https://adinasarapu.github.io/images/mstile-144x144.png?v=M44lzPylqQ">
<meta name="msapplication-config" content="https://adinasarapu.github.io/images/browserconfig.xml?v=M44lzPylqQ">
<meta name="theme-color" content="#ffffff">
<link rel="stylesheet" href="https://adinasarapu.github.io/assets/css/academicons.css"/>

<script type="text/x-mathjax-config"> MathJax.Hub.Config({ TeX: { equationNumbers: { autoNumber: "all" } } }); </script>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/latest.js?config=TeX-MML-AM_CHTML' async></script>

<!-- end custom head snippets -->

  </head>

  <body>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->
    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <button><div class="navicon"></div></button>
        <ul class="visible-links">
          <li class="masthead__menu-item masthead__menu-item--lg"><a href="https://adinasarapu.github.io/">Ashok R. Dinasarapu Ph.D</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/publications/">Publications</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/teaching/">Teaching</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/portfolio/">Big Data</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/year-archive/">NGS | Proteomics</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/cv/">Résumé</a></li>
          
        </ul>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>

    



<div id="main" role="main">
  


  <div class="sidebar sticky">
  



<div itemscope itemtype="http://schema.org/Person">

  <div class="author__avatar">
    
    	<img src="https://adinasarapu.github.io/images/profile.png" class="author__avatar" alt="Scientist, Bioinformatics">
    
  </div>

  <div class="author__content">
    <h3 class="author__name">Scientist, Bioinformatics</h3>
    <p class="author__bio">Bridging the gap between Genomics and Data Science</p>
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      
        <li><i class="fa fa-fw fa-map-marker" aria-hidden="true"></i> Atlanta, GA</li>
      
      
      
      
        <li><a href="mailto:ashok.reddy.dinasarapu@emory.edu"><i class="fas fa-fw fa-envelope" aria-hidden="true"></i> Email</a></li>
      
      
       
        <li><a href="https://www.researchgate.net/profile/Ashok_Dinasarapu"><i class="fab fa-fw fa-researchgate" aria-hidden="true"></i> ResearchGate</a></li>
      
      
        <li><a href="https://twitter.com/adinasarapu"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
      
      
      
      
        <li><a href="https://www.linkedin.com/in/dareddy"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i> LinkedIn</a></li>
      
      
      
      
      
        <li><a href="https://bitbucket.org/adinasarapu"><i class="fab fa-fw fa-bitbucket" aria-hidden="true"></i> Bitbucket</a></li>
      
      
        <li><a href="https://github.com/adinasarapu"><i class="fab fa-fw fa-github" aria-hidden="true"></i> Github</a></li>
      
      
      
      
      
      
      
      
      
      
      
      
      
      
        <li><a href="https://scholar.google.com/citations?user=b6GBykAAAAAJ"><i class="fas fa-fw fa-graduation-cap"></i> Google Scholar</a></li>
      
      
        <li><a href="https://pubmed.ncbi.nlm.nih.gov/?term=dinasarapu+ashok+OR+D+ashok+reddy&sort=pubdate"><i class="ai ai-pubmed-square ai-fw"></i> PubMed</a></li>
      
      
        <li><a href="http://orcid.org/0000-0002-1423-1518"><i class="ai ai-orcid-square ai-fw"></i> ORCID</a></li>
      
      
      
    </ul>
  </div>
</div>

  
  </div>


  <div class="archive">
    
      <h1 class="page__title">Posts by Tags</h1>
    
    
<!--
# Jekyll Group-By-Array 0.1.0
# https://github.com/mushishi78/jekyll-group-by-array
# © 2015 Max White <mushishi78@gmail.com>
# MIT License
-->

<!-- Initialize -->




<!-- Map -->


<!-- Flatten -->


<!-- Uniq -->



<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->





<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->






<!-- If not equal to previous then it must be unique as sorted -->


<!-- Push to group_names -->







<!-- group_items -->


<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->



<!-- Collect if contains -->



<!-- Push to group_items -->




  
  <h2 id="10xgenomics" class="archive__subtitle">10XGenomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" rel="permalink">Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-03-06T00:00:00-05:00">March 06, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">spaceranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing.</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell gene expression data analysis on Cluster (10X Genomics, Cell Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-05:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><em>Updated on May 03, 2021</em></p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="10plex" class="archive__subtitle">10plex</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="16s-rrna-amplicon" class="archive__subtitle">16s rRNA amplicon</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-05:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="atac-seq-assay-for-transposase-accessible-chromatin-with-high-throughput-sequencing" class="archive__subtitle">ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing)</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/12/blog-post-atacseq/" rel="permalink">ATAC-seq peak calling with MACS2
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-12-05T00:00:00-05:00">December 05, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing) is a next-generation sequencing approach for the analysis of open chromatin regions to assess the genome-wise chromatin accessibility.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="amazon-web-services" class="archive__subtitle">Amazon Web Services</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Updated on September 13, 2021</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="bwa" class="archive__subtitle">BWA</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/12/blog-post-atacseq/" rel="permalink">ATAC-seq peak calling with MACS2
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-12-05T00:00:00-05:00">December 05, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing) is a next-generation sequencing approach for the analysis of open chromatin regions to assess the genome-wise chromatin accessibility.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="bioinformatics" class="archive__subtitle">Bioinformatics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2022/10/blog-post-kaplan-meier-curve/" rel="permalink">Kaplan-Meier Curve using R
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2022-10-19T00:00:00-04:00">October 19, 2022</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Kaplan-Meier curve shows what the probability of an event (for example, survival) at a certain time interval. The log-rank test compares the survival curves of two or more groups. With a small subset of patients, the Kaplan-Meier estimates can be misleading and should be interpreted with caution.</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Updated on September 13, 2021</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="cell-ranger" class="archive__subtitle">Cell Ranger</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell gene expression data analysis on Cluster (10X Genomics, Cell Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-05:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><em>Updated on May 03, 2021</em></p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="chromium-single-cell-gene-expression" class="archive__subtitle">Chromium Single Cell Gene Expression</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell gene expression data analysis on Cluster (10X Genomics, Cell Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-05:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><em>Updated on May 03, 2021</em></p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="cluster-computing" class="archive__subtitle">Cluster Computing</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" rel="permalink">Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-03-06T00:00:00-05:00">March 06, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">spaceranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing.</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell gene expression data analysis on Cluster (10X Genomics, Cell Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-05:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><em>Updated on May 03, 2021</em></p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="dada2" class="archive__subtitle">DADA2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-05:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="emory-university" class="archive__subtitle">Emory University</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2023/02/blog-post-somalogic-proteomics/" rel="permalink">Quantitative proteomics: aptamer-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  5 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2023-02-20T00:00:00-05:00">February 20, 2023</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The aptamer-based <a href="https://somalogic.com">SomaScan®</a> assay is one of the popular methods of measuring abundances of protein targets. There is very little information on correlation between mass spectrometry (MS)-based proteomics, SOMAscan and Olink assays; Olink is another popular high throughput antibody-based platform. Some studies also reported a measurement variation between those platforms. In general aptamers/SOMAmers are selected against target proteins in their native conformation and in some cases against a functional protein with “known” post translational modifications (PTMs). It’s well known that novel PTMs (pathogen or disease-induced) can impact the protein structure, electrophilicity and interactions with proteins. The other main disadvantage is quantification which is based on DNA microarray chips (background noise). The main advantages are lower cost and data analysis.</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2022/10/blog-post-kaplan-meier-curve/" rel="permalink">Kaplan-Meier Curve using R
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2022-10-19T00:00:00-04:00">October 19, 2022</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Kaplan-Meier curve shows what the probability of an event (for example, survival) at a certain time interval. The log-rank test compares the survival curves of two or more groups. With a small subset of patients, the Kaplan-Meier estimates can be misleading and should be interpreted with caution.</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Updated on September 13, 2021</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="fastqc" class="archive__subtitle">FastQC</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-04:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="feature-barcoding" class="archive__subtitle">Feature Barcoding</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell gene expression data analysis on Cluster (10X Genomics, Cell Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-05:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><em>Updated on May 03, 2021</em></p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="gatk" class="archive__subtitle">GATK</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-05:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="genomics" class="archive__subtitle">Genomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="genotypes" class="archive__subtitle">Genotypes</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="homer" class="archive__subtitle">HOMER</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/12/blog-post-atacseq/" rel="permalink">ATAC-seq peak calling with MACS2
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-12-05T00:00:00-05:00">December 05, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing) is a next-generation sequencing approach for the analysis of open chromatin regions to assess the genome-wise chromatin accessibility.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="htseq" class="archive__subtitle">HTSeq</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-04:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="humann2" class="archive__subtitle">HUMAnN2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-04:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="indel" class="archive__subtitle">Indel</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-05:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="kaplan-meier-curve" class="archive__subtitle">Kaplan-Meier-curve</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2022/10/blog-post-kaplan-meier-curve/" rel="permalink">Kaplan-Meier Curve using R
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2022-10-19T00:00:00-04:00">October 19, 2022</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Kaplan-Meier curve shows what the probability of an event (for example, survival) at a certain time interval. The log-rank test compares the survival curves of two or more groups. With a small subset of patients, the Kaplan-Meier estimates can be misleading and should be interpreted with caution.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="lfq" class="archive__subtitle">LFQ</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Updated on September 13, 2021</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="macs2" class="archive__subtitle">MACS2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/12/blog-post-atacseq/" rel="permalink">ATAC-seq peak calling with MACS2
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-12-05T00:00:00-05:00">December 05, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>ATAC-seq (Assay for Transposase Accessible Chromatin with high-throughput Sequencing) is a next-generation sequencing approach for the analysis of open chromatin regions to assess the genome-wise chromatin accessibility.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="ms-based-proteomics" class="archive__subtitle">MS-based Proteomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2023/02/blog-post-somalogic-proteomics/" rel="permalink">Quantitative proteomics: aptamer-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  5 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2023-02-20T00:00:00-05:00">February 20, 2023</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The aptamer-based <a href="https://somalogic.com">SomaScan®</a> assay is one of the popular methods of measuring abundances of protein targets. There is very little information on correlation between mass spectrometry (MS)-based proteomics, SOMAscan and Olink assays; Olink is another popular high throughput antibody-based platform. Some studies also reported a measurement variation between those platforms. In general aptamers/SOMAmers are selected against target proteins in their native conformation and in some cases against a functional protein with “known” post translational modifications (PTMs). It’s well known that novel PTMs (pathogen or disease-induced) can impact the protein structure, electrophilicity and interactions with proteins. The other main disadvantage is quantification which is based on DNA microarray chips (background noise). The main advantages are lower cost and data analysis.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="ms3-spectra" class="archive__subtitle">MS3 spectra</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="matrixeqtl" class="archive__subtitle">MatrixEqtl</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-05:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="maxquant" class="archive__subtitle">MaxQuant</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Updated on September 13, 2021</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="metaphlan2" class="archive__subtitle">MetaPhlAn2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-04:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="ngs" class="archive__subtitle">NGS</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="otu" class="archive__subtitle">OTU</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-05:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="olink-proteomics" class="archive__subtitle">Olink Proteomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2023/02/blog-post-somalogic-proteomics/" rel="permalink">Quantitative proteomics: aptamer-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  5 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2023-02-20T00:00:00-05:00">February 20, 2023</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The aptamer-based <a href="https://somalogic.com">SomaScan®</a> assay is one of the popular methods of measuring abundances of protein targets. There is very little information on correlation between mass spectrometry (MS)-based proteomics, SOMAscan and Olink assays; Olink is another popular high throughput antibody-based platform. Some studies also reported a measurement variation between those platforms. In general aptamers/SOMAmers are selected against target proteins in their native conformation and in some cases against a functional protein with “known” post translational modifications (PTMs). It’s well known that novel PTMs (pathogen or disease-induced) can impact the protein structure, electrophilicity and interactions with proteins. The other main disadvantage is quantification which is based on DNA microarray chips (background noise). The main advantages are lower cost and data analysis.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="perseus" class="archive__subtitle">Perseus</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Updated on September 13, 2021</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="proteomics" class="archive__subtitle">Proteomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Updated on September 13, 2021</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="quantitative-proteomics" class="archive__subtitle">Quantitative Proteomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2023/02/blog-post-somalogic-proteomics/" rel="permalink">Quantitative proteomics: aptamer-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  5 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2023-02-20T00:00:00-05:00">February 20, 2023</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The aptamer-based <a href="https://somalogic.com">SomaScan®</a> assay is one of the popular methods of measuring abundances of protein targets. There is very little information on correlation between mass spectrometry (MS)-based proteomics, SOMAscan and Olink assays; Olink is another popular high throughput antibody-based platform. Some studies also reported a measurement variation between those platforms. In general aptamers/SOMAmers are selected against target proteins in their native conformation and in some cases against a functional protein with “known” post translational modifications (PTMs). It’s well known that novel PTMs (pathogen or disease-induced) can impact the protein structure, electrophilicity and interactions with proteins. The other main disadvantage is quantification which is based on DNA microarray chips (background noise). The main advantages are lower cost and data analysis.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="rna-seq" class="archive__subtitle">RNA-Seq</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-04:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-05:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-05:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="rnaseq" class="archive__subtitle">RNASeq</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-04:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="sge-cluster" class="archive__subtitle">SGE Cluster</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" rel="permalink">Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-03-06T00:00:00-05:00">March 06, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">spaceranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="snp" class="archive__subtitle">SNP</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-05:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="somalogic" class="archive__subtitle">SOMALogic</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2023/02/blog-post-somalogic-proteomics/" rel="permalink">Quantitative proteomics: aptamer-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  5 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2023-02-20T00:00:00-05:00">February 20, 2023</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The aptamer-based <a href="https://somalogic.com">SomaScan®</a> assay is one of the popular methods of measuring abundances of protein targets. There is very little information on correlation between mass spectrometry (MS)-based proteomics, SOMAscan and Olink assays; Olink is another popular high throughput antibody-based platform. Some studies also reported a measurement variation between those platforms. In general aptamers/SOMAmers are selected against target proteins in their native conformation and in some cases against a functional protein with “known” post translational modifications (PTMs). It’s well known that novel PTMs (pathogen or disease-induced) can impact the protein structure, electrophilicity and interactions with proteins. The other main disadvantage is quantification which is based on DNA microarray chips (background noise). The main advantages are lower cost and data analysis.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="spatial-gene-expression" class="archive__subtitle">SPATIAL GENE EXPRESSION</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" rel="permalink">Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-03-06T00:00:00-05:00">March 06, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">spaceranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="star" class="archive__subtitle">STAR</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-04:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/01/blog-post-gatk/" rel="permalink">Genomic variants from RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-05T00:00:00-05:00">January 05, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of genomic variation in expressed regions.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="seurat" class="archive__subtitle">Seurat</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell gene expression data analysis on Cluster (10X Genomics, Cell Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-05:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><em>Updated on May 03, 2021</em></p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="single-cell" class="archive__subtitle">Single Cell</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell gene expression data analysis on Cluster (10X Genomics, Cell Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-05:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><em>Updated on May 03, 2021</em></p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="single-cell-rna-sequencing-scrna-seq" class="archive__subtitle">Single cell RNA-sequencing (scRNA-seq)</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-sc-ranseq/" rel="permalink">Single cell gene expression data analysis on Cluster (10X Genomics, Cell Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  7 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-11-18T00:00:00-05:00">November 18, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p><em>Updated on May 03, 2021</em></p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="space-ranger" class="archive__subtitle">Space Ranger</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" rel="permalink">Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-03-06T00:00:00-05:00">March 06, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">spaceranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="survival-analysis" class="archive__subtitle">Survival analysis</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2022/10/blog-post-kaplan-meier-curve/" rel="permalink">Kaplan-Meier Curve using R
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2022-10-19T00:00:00-04:00">October 19, 2022</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Kaplan-Meier curve shows what the probability of an event (for example, survival) at a certain time interval. The log-rank test compares the survival curves of two or more groups. With a small subset of patients, the Kaplan-Meier estimates can be misleading and should be interpreted with caution.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="tmt" class="archive__subtitle">TMT</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="tmt-10plex" class="archive__subtitle">TMT 10plex</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="vcf" class="archive__subtitle">VCF</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="vep" class="archive__subtitle">VEP</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="variant-calling" class="archive__subtitle">Variant calling</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="visium-spatial-gene-expression-solution" class="archive__subtitle">Visium Spatial Gene Expression Solution</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" rel="permalink">Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-03-06T00:00:00-05:00">March 06, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">spaceranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="vital" class="archive__subtitle">Vital</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2022/10/blog-post-kaplan-meier-curve/" rel="permalink">Kaplan-Meier Curve using R
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2022-10-19T00:00:00-04:00">October 19, 2022</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Kaplan-Meier curve shows what the probability of an event (for example, survival) at a certain time interval. The log-rank test compares the survival curves of two or more groups. With a small subset of patients, the Kaplan-Meier estimates can be misleading and should be interpreted with caution.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="wgs" class="archive__subtitle">WGS</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="aptamer-based-proteomics" class="archive__subtitle">aptamer-based proteomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2023/02/blog-post-somalogic-proteomics/" rel="permalink">Quantitative proteomics: aptamer-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  5 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2023-02-20T00:00:00-05:00">February 20, 2023</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The aptamer-based <a href="https://somalogic.com">SomaScan®</a> assay is one of the popular methods of measuring abundances of protein targets. There is very little information on correlation between mass spectrometry (MS)-based proteomics, SOMAscan and Olink assays; Olink is another popular high throughput antibody-based platform. Some studies also reported a measurement variation between those platforms. In general aptamers/SOMAmers are selected against target proteins in their native conformation and in some cases against a functional protein with “known” post translational modifications (PTMs). It’s well known that novel PTMs (pathogen or disease-induced) can impact the protein structure, electrophilicity and interactions with proteins. The other main disadvantage is quantification which is based on DNA microarray chips (background noise). The main advantages are lower cost and data analysis.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="brightfield-microscope-image" class="archive__subtitle">brightfield microscope image</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" rel="permalink">Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-03-06T00:00:00-05:00">March 06, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Running <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">spaceranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="cnv" class="archive__subtitle">cnv</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/09/blog-post-trio/" rel="permalink">Copy number variation discovery workflows using NGS  data.
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-19T00:00:00-04:00">August 19, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Copy number variations (CNVs) represent gain or loss of genomic regions. CNVs transmit from parents to offspring or arise <i>de novo</i> and play important role in neuro-psychiatric disorders and cancers.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="computing" class="archive__subtitle">computing</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-05:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/09/blog-post-trio/" rel="permalink">Copy number variation discovery workflows using NGS  data.
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-19T00:00:00-04:00">August 19, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Copy number variations (CNVs) represent gain or loss of genomic regions. CNVs transmit from parents to offspring or arise <i>de novo</i> and play important role in neuro-psychiatric disorders and cancers.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="curation" class="archive__subtitle">curation</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/publication/2013-ucsd-sgmp" rel="permalink">A comprehensive map of the human complement system
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-20T00:00:00-04:00">August 20, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The human complement activation takes place through one or more of the well-established pathways consisting of plasma and membrane-bound proteins.</p>
</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="discordant-sex-informaion" class="archive__subtitle">discordant sex informaion</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-05:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="eqtl" class="archive__subtitle">eQTL</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-05:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="edger" class="archive__subtitle">edgeR</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/03/blog-post-rnaseq/" rel="permalink">RNA-Seq: raw reads to differential expression
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-03-20T00:00:00-04:00">March 20, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>A simple RNA-Seq differential expression analysis using High Performance Computing (HPC).</p>

</p>
    
    
    

  </article>
</div>

  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-05:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="functional-profiling" class="archive__subtitle">functional profiling</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-04:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="genomics" class="archive__subtitle">genomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/09/blog-post-trio/" rel="permalink">Copy number variation discovery workflows using NGS  data.
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-19T00:00:00-04:00">August 19, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Copy number variations (CNVs) represent gain or loss of genomic regions. CNVs transmit from parents to offspring or arise <i>de novo</i> and play important role in neuro-psychiatric disorders and cancers.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="heterozygosity-rate" class="archive__subtitle">heterozygosity rate</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-05:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="htseq" class="archive__subtitle">htseq</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-eqtl/" rel="permalink">eQTL analysis of RNA-Seq data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-17T00:00:00-05:00">December 17, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.<br />
Based on the hg38 reference genome, paired-end reads are mapped by <a href="https://academic.oup.com/bioinformatics/article/29/1/15/272537">STAR</a> aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by <a href="https://htseq.readthedocs.io/en/release_0.9.1/">HTSeq</a> that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/">edgeR</a>.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="human-complement" class="archive__subtitle">human complement</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/publication/2013-ucsd-sgmp" rel="permalink">A comprehensive map of the human complement system
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-20T00:00:00-04:00">August 20, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The human complement activation takes place through one or more of the well-established pathways consisting of plasma and membrane-bound proteins.</p>
</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="ibac" class="archive__subtitle">iBAC</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Updated on September 13, 2021</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="isobaric-labeling" class="archive__subtitle">isobaric labeling</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="label-free-quantitation" class="archive__subtitle">label free quantitation</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" rel="permalink">Quantitative proteomics: label-free quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Updated on September 13, 2021</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="metagenomics" class="archive__subtitle">metagenomics</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-04:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="microbiome" class="archive__subtitle">microbiome</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-05:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="missingness" class="archive__subtitle">missingness</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-05:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="networks" class="archive__subtitle">networks</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/publication/2013-ucsd-sgmp" rel="permalink">A comprehensive map of the human complement system
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  less than 1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-08-20T00:00:00-04:00">August 20, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The human complement activation takes place through one or more of the well-established pathways consisting of plasma and membrane-bound proteins.</p>
</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="plink" class="archive__subtitle">plink</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-05:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="qiime2" class="archive__subtitle">qiime2</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2019/01/blog-post-qiime2/" rel="permalink">Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00-05:00">January 01, 2019</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The 16S ribosomal RNA (rRNA) gene of Bacteria codes for the RNA component of the 30S subunit. Different bacterial species have one to multiple copies of the 16S rRNA gene, and each with 9 hypervariable regions, V1-V9. High-throughput sequencing of 16S rRNA gene (a “marker gene”) amplicons has become a widely used method to study bacterial phylogeny and species classification.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="runs-of-homozygosity" class="archive__subtitle">runs of homozygosity</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2017/12/blog-post-plink/" rel="permalink">Quality control for GWAS studies
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-16T00:00:00-05:00">December 16, 2017</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>An important step in the analysis of genome-wide association studies (GWAS) is to identify problematic subjects and markers. Quality control  (QC) in GWAS removes markers and individuals, and greatly increases the accuracy of findings.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="shotgun-metagenomics-sequencing" class="archive__subtitle">shotgun metagenomics sequencing</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-04:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="tandem-mass-tag" class="archive__subtitle">tandem mass tag</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

  

  
  <h2 id="taxonomic-profiling" class="archive__subtitle">taxonomic profiling</h2>
  
    





<div class="list__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2018/07/blog-post-metagenomics/" rel="permalink">Taxonomic and functional profiling of the microbiome - whole genome shotgun metagenomics
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  1 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-07-27T00:00:00-04:00">July 27, 2018</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>This workflow consists of taxonomic and functional profiling of shotgun metagenomics sequencing (MGS) reads using MetaPhlAn2 and HUMAnN2, respectively. To perform taxonomic (phyla, genera or species level) profiling of the MGS data, the MetaPhlAn2 pipeline was run on a high performance multicore cluster computing environment.</p>

</p>
    
    
    

  </article>
</div>

  

  </div>
</div>


    <div class="page__footer">
      <footer>
        <!-- start custom footer snippets -->
<!--<a href="/cv/">Resume</a> -->
<!-- end custom footer snippets -->

        

<div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    
    
    
    
      <li><a href="http://github.com/adinasarapu"><i class="fab fa-github" aria-hidden="true"></i> GitHub</a></li>
    
    
      <li><a href="http://bitbucket.org/adinasarapu"><i class="fab fa-bitbucket" aria-hidden="true"></i> Bitbucket</a></li>
    
    <!--<li><a href="https://adinasarapu.github.io/feed.xml">
	<i class="fa fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>-->
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2023 Ashok R. Dinasarapu. 101 Woodruff Circle, Woodruff Memorial Research Building,  Room#6302, Emory University, Atlanta, GA 30322.</div>

      </footer>
    </div>

    <script src="https://adinasarapu.github.io/assets/js/main.min.js"></script>




  <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'G-RWM39QLMPF', 'auto');
  ga('send', 'pageview');
</script>






  </body>
</html>

