---
title: 'eQTL analysis of RNA-Seq data'
date: 2017-12-17
permalink: /posts/2017/12/blog-post-eqtl/
tags:
  - eQTL
  - RNA-Seq
  - htseq
  - MatrixEqtl
  - edgeR
---
Genetic locus that affects gene expression is often referred to as expression quantitative trait locus (eQTL). eQTL mapping studies assesses the association of SNPs with genome-wide expression levels.  
Based on the hg38 reference genome, paired-end reads are mapped by [STAR](https://academic.oup.com/bioinformatics/article/29/1/15/272537) aligner. The mapped reads are used for expression quantification without assembling transcripts by counting the number of reads that map to an exon by [HTSeq](https://htseq.readthedocs.io/en/release_0.9.1/) that uses Refseq gene annotations. Then, to correct for systematic variability such as library fragment size, sequence composition bias, and read depth the raw counts are normalized as trimmed mean of M-values (TMM) through [edgeR](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/).  

Beyond quantifying gene expression, the data generated by RNA-Seq facilitate the integration of expression data with genotyping data also known as expression quantitative trait loci (eQTLs) analysis. Infinium CytoSNP-850K v1.2 arrays are used to identify genetic and structural variations. 
The array data is analyzed using GenomeStudioR or BlueFuse Multi software based on the reference human genome (hg38/GRCh38). After loading the raw array data, the SNP manifest file (*.bpm), and standard cluster file (*.egt) are imported into GenomeStudio and clustering of intensities for SNPs are performed. Genotyping calls for a specific DNA made by the calling algorithm (GenCall) which relies on information provided by the GenTrain clustering algorithm.  

[Matrix eQTL](http://www.bios.unc.edu/research/genomic_software/Matrix_eQTL/) was used to efficiently test the associations by modeling the effect of genotype as additive linear.  
![eqtl_figure](/images/DEG.png)  

Data analysis pipeline for RNA-Seq based eQTL mapping:   
- [Quality control](https://bitbucket.org/adinasarapu/clustercomputing/src/6bad66b2e4ce1f98d3392cb4af7cbf37bb08f73f/job_fastqc.sh)
- [Read mapping](https://bitbucket.org/adinasarapu/clustercomputing/src/6bad66b2e4ce1f98d3392cb4af7cbf37bb08f73f/job_star_mapping.sh)
- [Post mapping QC](https://bitbucket.org/adinasarapu/clustercomputing/src/a9ecd1df38811b6078c2397476dd12a48ac18a50/job_post_mapping_qc.sh)
- [HTSeq-count](https://bitbucket.org/adinasarapu/clustercomputing/src/a9ecd1df38811b6078c2397476dd12a48ac18a50/job_htseq_count.sh)
- [Normalization, Differential expression](https://bitbucket.org/adinasarapu/clustercomputing/src/a9ecd1df38811b6078c2397476dd12a48ac18a50/HTseqCountToANOVA.R)
- [Create SNP Matrix](https://bitbucket.org/adinasarapu/clustercomputing/src/8a3ba3ef7eaa3937b04b101a843e2a5fed88e52b/ExtractMultiSampleGenotype.R)
- [Matrix eQTL](https://bitbucket.org/adinasarapu/clustercomputing/src/a1e0bf858af1619edae4816b03735aea9ab215aa/MatrixEQTL_CisTrans.R)
- [Manhattan plot](https://bitbucket.org/adinasarapu/clustercomputing/src/dd0755abdece34f8e1f8f46329f2bd570bfd0318/ManhattanPlot_GWAS.R)
