---
title: 'Bioinformatic processing of microbiome (16s rRNA gene) sequencing data - QIIME 1.8.0'
date: 2017-08-20
permalink: /posts/2017/09/blog-post-qiime1/
tags:
  - microbiome
  - computing
  - qiime1
---
The first step in this pipeline is to join forward and reverse reads using PANDAseq. Then format the joined sequence files through by adding QIIME headers with the respective sample ID to each sequence before concatenating into one file for input into QIIME 1.8.0. Operational taxonomic units (OTUs) will be picked using the QIIME pick_closed_reference_otus.py script with a threshold of 97% identity to the Greengenes database. The Shannon alpha diversity will be calculated on the unfiltered biom table using the alpha_diversity.py script, and weighted UniFrac distances are calculated with the beta_diversity.py script.
| [QIIME1](https://bitbucket.org/adinasarapu/clustercomputing/src/6e3396384fc31cdb4703534a4ca42d7f6a979954/qiime1.py) |
