---
title: 'RNA-Seq eQTL Analysis Pipeline: Uncovering Genetic Influences on Gene Expression'
date: 2017-12-17
permalink: /posts/2017/12/blog-post-eqtl/
tags:
  - eQTL
  - RNA-Seq
  - htseq
  - MatrixEqtl
  - edgeR
---  
Understanding how genetic variations impact gene expression is crucial for uncovering the underlying mechanisms of complex traits and diseases. One powerful tool for this investigation is the expression quantitative trait locus (eQTL) analysis, which examines the relationship between genetic variants and genome-wide expression levels.  

Our eQTL analysis pipeline is built on the hg38 reference genome and employs several key steps:  
<ol>
<li><b>Mapping Reads with STAR</b>: Paired-end reads from RNA-Seq data are aligned using the STAR aligner. STAR is known for its speed and accuracy in mapping reads to the reference genome.</li>

<li><b>Quantifying Gene Expression with HTSeq</b>: Following alignment, we quantify gene expression by counting the number of reads mapping to exons using HTSeq. This step utilizes RefSeq gene annotations to ensure precise measurement of expression levels without the need for transcript assembly.</li>

<li><b>Normalizing Data with edgeR</b>: To account for systematic variability such as library fragment size, sequence composition bias, and read depth, we normalize the raw counts using the trimmed mean of M-values (TMM) approach through edgeR. This normalization helps in making accurate comparisons across samples.</li>

<li><b>Genotyping Data</b>: In addition to gene expression quantification, RNAseq data can be combined with genotyping data for eQTL analysis. We use Infinium CytoSNP-850K v1.2 arrays to detect genetic and structural variations.</li>

<li><b>Analyzing Array Data</b>: The array data is processed using GenomeStudioR or BlueFuse Multi software based on the hg38/GRCh38 reference genome. After importing the raw array data along with the SNP manifest file (.bpm) and standard cluster file (.egt) into GenomeStudio, we perform clustering of SNP intensities. Genotyping calls are made using the GenCall algorithm, which is informed by the GenTrain clustering algorithm.</li>
</ol>

[Matrix eQTL](http://www.bios.unc.edu/research/genomic_software/Matrix_eQTL/) was used to efficiently test the associations by modeling the effect of genotype as additive linear.  
![eqtl_figure](/images/DEG.png)  

Data analysis pipeline for RNA-Seq based eQTL mapping:   
- [Quality control](https://bitbucket.org/adinasarapu/clustercomputing/src/6bad66b2e4ce1f98d3392cb4af7cbf37bb08f73f/job_fastqc.sh)
- [Read mapping](https://bitbucket.org/adinasarapu/clustercomputing/src/6bad66b2e4ce1f98d3392cb4af7cbf37bb08f73f/job_star_mapping.sh)
- [Post mapping QC](https://bitbucket.org/adinasarapu/clustercomputing/src/a9ecd1df38811b6078c2397476dd12a48ac18a50/job_post_mapping_qc.sh)
- [HTSeq-count](https://bitbucket.org/adinasarapu/clustercomputing/src/a9ecd1df38811b6078c2397476dd12a48ac18a50/job_htseq_count.sh)
- [Normalization, Differential expression](https://bitbucket.org/adinasarapu/clustercomputing/src/a9ecd1df38811b6078c2397476dd12a48ac18a50/HTseqCountToANOVA.R)
- [Create SNP Matrix](https://bitbucket.org/adinasarapu/clustercomputing/src/8a3ba3ef7eaa3937b04b101a843e2a5fed88e52b/ExtractMultiSampleGenotype.R)
- [Matrix eQTL](https://bitbucket.org/adinasarapu/clustercomputing/src/a1e0bf858af1619edae4816b03735aea9ab215aa/MatrixEQTL_CisTrans.R)
- [Manhattan plot](https://bitbucket.org/adinasarapu/clustercomputing/src/dd0755abdece34f8e1f8f46329f2bd570bfd0318/ManhattanPlot_GWAS.R)
