---
title: 'Bioinformatic processing of microbiome (16s rRNA gene) sequencing data - QIIME 2 (2017.7)'
date: 2017-08-27
permalink: /posts/2017/09/blog-post-qiime2/
tags:
  - microbiome
  - computing
  - qiime2
---
QIIME 2 (2017.7) is a powerful, and community developed, microbiome analysis package that enables researchers to start an analysis with raw 16s rRNA sequence data. Data produced by QIIME 2 exist as QIIME 2 artifacts. A QIIME 2 artifact typically has the .qza file extension when output data stored in a file. Visualizations are another type of data (.qzv file) generated by QIIME 2, which can be viewed using a web interface [https://view.qiime2.org](https://view.qiime2.org) (at Firefox web browser) without requiring a QIIME installation. Since QIIME 2 works with artifacts instead of data files (e.g. FASTA files), we must create a QIIME 2 artifact by importing our fastq.gz data files. 

Execute the following three steps:
1. Find PHRED offset; i.e fastq quality score format [script](https://bitbucket.org/adinasarapu/clustercomputing/src/6e3396384fc31cdb4703534a4ca42d7f6a979954/fastq_manifest_phred.sh).
2. Create a sample manifest file, import FASTQ files as QIIME 2 artifact and plot for seq positional qualities [script](https://bitbucket.org/adinasarapu/clustercomputing/src/6e3396384fc31cdb4703534a4ca42d7f6a979954/qiime2.2017.7_step1.sh).
3. To denoise-paired data, filter chimeras and to export data as normalized BIOM format [script](https://bitbucket.org/adinasarapu/clustercomputing/src/6e3396384fc31cdb4703534a4ca42d7f6a979954/qiime2.2017.7_step2.sh).

These scripts produce a key summary table or BIOM (Biological Observation Matrix) file containing feature (Operational Taxonomic Units, OTUs) abundance information across samples, along with various annotations and sample metadata. 

Upload BIOM file to [MicrobiomeAnalyst](http://www.microbiomeanalyst.ca), a web-based tool for comprehensive statistical, visual and meta-analysis of microbiome data, for <b>taxonomic profiling</b> - to characterize community compositions based on methods developed in ecology such as alpha-diversity (within-sample diversity) or beta-diversity (between-sample diversity) and <b>comparative analysis</b> - to identify features that are significantly different among conditions under study.  
