---
title: 'Variant calling using RNA-Seq data'
date: 2018-01-05
permalink: /posts/2017/12/blog-post-gatk/
tags:
  - GATK
  - RNA-Seq
  - STAR
  - SNP
  - Indel
---
RNA-Seq allows the detection and quantification of known and rare RNA transcripts within a sample. In addition to differential expression and detection of novel transcripts, RNA-seq also supports the detection of mutations and germline variation for hundreds to thousands of expressed genetic variants. 

Currently few workflows exist for detecting SNPs in RNA-seq data, including [eSNV-detect](http://bioinformaticstools.mayo.edu/research/esnv-detect/), [SNPiR](https://github.com/rpiskol/SNPiR) and [Opossum](https://github.com/pysam-developers/pysam). Here, I have employed [GATK workflow for SNP and indel calling on RNAseq data](https://software.broadinstitute.org/gatk/documentation/article.php?id=3891), which is based on the following steps: 
- Reference (hg38) based read mapping using STAR aligner. This is a STAR 2-pass approach with the suggested parameters. In this STAR 2-pass approach, splice junctions detected in a first alignment run are used to guide the final alignment.
- Add read group information, sort, mark the duplicates and index with picard.jar. 
- GATK SplitNCigarReads, split the reads into exon segments (removing Ns but maintaining grouping information) and reassigning mapping qualities.
- Indel realignment and recalibration of Base qualities and
- Variant calling with GATK HaplotypeCaller, and finally filtering the variants with GATK VariantFiltration.

My qsub-based pipeline is available at [bitbucket.org](https://bitbucket.org/adinasarapu/clustercomputing/src/b888c564a8d9f0c7f433086f2025b93826a49fe8/job_rnaseq_variant_caller.sh)

