---
title: 'Chromium single cell 3â€² RNA-seq data analysis using Cell Ranger and Seurat on Cluster'
date: 2019-01-06
permalink: /posts/2019/01/blog-post-sc-ranseq/
tags:
  - 10XGenomics
  - Single cell RNA-sequencing (scRNA-seq)
  - Single Cell
  - Cell Ranger
  - Seurat
  - Cluster Computing

---
Running [cellranger](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger) at HGCC (Human Genetics Computing Cluster) cluster that uses Sun Grid Engine (SGE) queuing system to run via batch scheduling. This allows highly parallelizable jobs to utilize multiple cores concurrently, dramatically reducing time to solution.

There are 4 steps to analyze Chromium Single Cell data.

**Step 1**: `cellranger mkfastq` demultiplexes raw base call (BCL) files generated by Illumina sequencers into FASTQ files.  
**Step 2**: `cellranger count` takes FASTQ files from cellranger mkfastq and performs alignment, filtering, barcode counting, and UMI counting.  
**Step 3**: `cellranger aggr` aggregates outputs from multiple runs of cellranger count.  
**Step 4**: Downstream/Secondary analysis using R package [Seurat v3.0](https://satijalab.org/seurat/).

Running pipelines at our HGCC cluster requires the following:  

**1**. Load Cell Ranger module (`cellranger-3.0.1`) or install at `$HOME` directory and add PATH in `~/.bashrc`.  

**2**. Update Job config file (`cellranger-3.0.1/martian-cs/v3.1.0/jobmanagers/config.json`) threads and memory.

`"threads_per_job": 8,`  
`"memGB_per_job": 64,`

**3**. Update Template file (`cellranger-3.0.1/martian-cs/v3.1.0/jobmanagers/sge.template`).

`#!/bin/bash`  
`#$ -pe smp __MRO_THREADS__`  
`#$ -q b.q`  
`#$ -S /bin/bash`  
`#$ -m abe`  
`#$ -M <e-mail>`  

**4**. Download single cell gene expression and reference genome datasets from [10XGenomics](https://www.10xgenomics.com/resources/datasets/).  

**5**. Create `sge.sh` file  

`FASTQS="/sw/hgcc/Data/10XGenomics/pbmc_10k_v3_fastqs"`  
`TR="/sw/hgcc/Data/10XGenomics/refdata-cellranger-GRCh38-3.0.0"`  

`cellranger count --disable-ui \`  
  `--id=PBMC_10k_v3 \`  
  `--transcriptome=${TR} \`  
  `--fastqs=${FASTQS} \`  
  `--sample=pbmc_10k_v3 \`  
  `--expect-cells=10000 \`  
  `--jobmode=sge --mempercore=8`  

**6**. Run  

`bash sge.sh`

---

[^1]: [10XGenomics](https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome)
[^2]: [Seurat](https://satijalab.org/seurat/)
