

<!doctype html>
<html lang="en" class="no-js">
  <head>
    

<meta charset="utf-8">



<!-- begin SEO -->









<title>Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster - Ashok R. Dinasarapu</title>







<meta property="og:locale" content="en-US">
<meta property="og:site_name" content="Ashok R. Dinasarapu">
<meta property="og:title" content="Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster">


  <link rel="canonical" href="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/">
  <meta property="og:url" content="http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/">



  <meta property="og:description" content="Running cellranger as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.">





  

  





  <meta property="og:type" content="article">
  <meta property="article:published_time" content="2019-11-18T00:00:00-08:00">








  <script type="application/ld+json">
    {
      "@context" : "http://schema.org",
      "@type" : "Person",
      "name" : "Ashok R. Dinasarapu",
      "url" : "http://localhost:4000",
      "sameAs" : null
    }
  </script>






<!-- end SEO -->


<link href="http://localhost:4000/feed.xml" type="application/atom+xml" rel="alternate" title="Ashok R. Dinasarapu Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="http://localhost:4000/assets/css/main.css">

<meta http-equiv="cleartype" content="on">
    

<!-- start custom head snippets -->

<link rel="apple-touch-icon" sizes="57x57" href="http://localhost:4000/images/apple-touch-icon-57x57.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="60x60" href="http://localhost:4000/images/apple-touch-icon-60x60.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="72x72" href="http://localhost:4000/images/apple-touch-icon-72x72.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="76x76" href="http://localhost:4000/images/apple-touch-icon-76x76.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="114x114" href="http://localhost:4000/images/apple-touch-icon-114x114.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="120x120" href="http://localhost:4000/images/apple-touch-icon-120x120.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="144x144" href="http://localhost:4000/images/apple-touch-icon-144x144.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="152x152" href="http://localhost:4000/images/apple-touch-icon-152x152.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="180x180" href="http://localhost:4000/images/apple-touch-icon-180x180.png?v=M44lzPylqQ">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-32x32.png?v=M44lzPylqQ" sizes="32x32">
<link rel="icon" type="image/png" href="http://localhost:4000/images/android-chrome-192x192.png?v=M44lzPylqQ" sizes="192x192">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-96x96.png?v=M44lzPylqQ" sizes="96x96">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-16x16.png?v=M44lzPylqQ" sizes="16x16">
<link rel="manifest" href="http://localhost:4000/images/manifest.json?v=M44lzPylqQ">
<link rel="mask-icon" href="http://localhost:4000/images/safari-pinned-tab.svg?v=M44lzPylqQ" color="#000000">
<link rel="shortcut icon" href="/images/favicon.ico?v=M44lzPylqQ">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-TileImage" content="http://localhost:4000/images/mstile-144x144.png?v=M44lzPylqQ">
<meta name="msapplication-config" content="http://localhost:4000/images/browserconfig.xml?v=M44lzPylqQ">
<meta name="theme-color" content="#ffffff">
<link rel="stylesheet" href="http://localhost:4000/assets/css/academicons.css"/>
<!-- end custom head snippets -->

  </head>

  <body>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->
    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <button><div class="navicon"></div></button>
        <ul class="visible-links">
          <li class="masthead__menu-item masthead__menu-item--lg"><a href="http://localhost:4000/">Ashok R. Dinasarapu</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/publications/">Publications</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/teaching/">Teaching</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/year-archive/">NGS | Proteomics</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/big-data/">BigData</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/cv/">Résumé</a></li>
          
        </ul>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>

    





<div id="main" role="main">
  


  <div class="sidebar sticky">
  



<div itemscope itemtype="http://schema.org/Person">

  <div class="author__avatar">
    
    	<img src="http://localhost:4000/images/profile.png" class="author__avatar" alt="Scientist, Bioinformatics">
    
  </div>

  <div class="author__content">
    <h3 class="author__name">Scientist, Bioinformatics</h3>
    <p class="author__bio">Emory University</p>
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      
        <li><i class="fa fa-fw fa-map-marker" aria-hidden="true"></i> Atlanta, GA</li>
      
      
        <li><i class="fa fa-fw fa-map-marker" aria-hidden="true"></i> </li>
      
      
      
        <li><a href="mailto:ashok.reddy.dinasarapu@emory.edu"><i class="fa fa-fw fa-envelope-square" aria-hidden="true"></i> Email</a></li>
      
      
      
        <li><a href="https://twitter.com/adinasarapu"><i class="fa fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
      
      
      
      
        <li><a href="https://www.linkedin.com/in/dareddy"><i class="fa fa-fw fa-linkedin-square" aria-hidden="true"></i> LinkedIn</a></li>
      
      
      
      
      
        <li><a href="https://bitbucket.org/adinasarapu"><i class="fa fa-fw fa-bitbucket" aria-hidden="true"></i> Bitbucket</a></li>
      
      
        <li><a href="https://github.com/adinasarapu"><i class="fa fa-fw fa-github" aria-hidden="true"></i> Github</a></li>
      
      
      
      
      
      
      
      
      
      
      
      
      
      
        <li><a href="https://scholar.google.com/citations?user=b6GBykAAAAAJ"><i class="ai ai-google-scholar-square ai-fw"></i> Google Scholar</a></li>
      
      
        <li><a href="http://orcid.org/0000-0002-1423-1518"><i class="ai ai-orcid-square ai-fw"></i> ORCID</a></li>
      
      
    </ul>
  </div>
</div>

  
  </div>


  <article class="page" itemscope itemtype="http://schema.org/CreativeWork">
    <meta itemprop="headline" content="Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster">
    <meta itemprop="description" content="Running cellranger as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.">
    <meta itemprop="datePublished" content="November 18, 2019">
    

    <div class="page__inner-wrap">
      
        <header>
          <h1 class="page__title" itemprop="headline">Single cell RNA-seq data analysis using CellRanger and Seurat on Cluster
</h1>
          
            <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  3 minute read
	
</p>
          
        
        
        
          <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Updated:</strong> <time datetime="2019-11-18T00:00:00-08:00">November 18, 2019</time></p>
        
        
             
        
    
        </header>
      

      <section class="page__content" itemprop="text">
        <p>Running <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">cellranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing system allows highly parallelizable jobs.</p>

<p>There are 4 steps to analyze Chromium Single Cell data<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>.</p>

<p><strong>Step 1</strong>: <code class="highlighter-rouge">cellranger mkfastq</code> demultiplexes raw base call (BCL) files generated by Illumina sequencers into FASTQ files.<br />
<strong>Step 2</strong>: <code class="highlighter-rouge">cellranger count</code> takes FASTQ files from cellranger mkfastq and performs alignment, filtering, barcode counting, and UMI counting. When doing large studies involving multiple GEM wells, run cellranger count on FASTQ data from each of the GEM wells individually, and then pool the results using cellranger aggr, as described <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/aggregate">here</a>.<br />
<strong>Step 3</strong>: <code class="highlighter-rouge">cellranger aggr</code> aggregates outputs from multiple runs of cellranger count.<br />
<strong>Step 4</strong>: Downstream/Secondary analysis using R package <a href="https://satijalab.org/seurat/">Seurat v3.0</a><sup id="fnref:2"><a href="#fn:2" class="footnote">2</a></sup>.</p>

<p>Running pipelines on cluster requires the following:</p>

<p><strong>1</strong>. Load Cell Ranger module (<code class="highlighter-rouge">cellranger-3.1.0</code>)<sup id="fnref:1:1"><a href="#fn:1" class="footnote">1</a></sup> or, download and uncompress cellranger at your <code class="highlighter-rouge">$HOME</code> directory and add PATH in <code class="highlighter-rouge">~/.bashrc</code>.</p>

<p><strong>2</strong>. Update job config file (<code class="highlighter-rouge">cellranger-3.1.0/martian-cs/v3.2.3/jobmanagers/config.json</code>) for threads and memory. For example</p>

<p><code class="highlighter-rouge">"threads_per_job": 20,</code><br />
<code class="highlighter-rouge">"memGB_per_job": 150,</code></p>

<p><strong>3</strong>. Update template file (<code class="highlighter-rouge">cellranger-3.1.0/martian-cs/v3.2.3/jobmanagers/sge.template</code>).</p>

<p><code class="highlighter-rouge"><span class="c">#!/bin/bash</span></code><br />
<code class="highlighter-rouge">#$ -pe smp __MRO_THREADS__</code><br />
<code class="highlighter-rouge">##$ -l mem_free=__MRO_MEM_GB__G</code> (comment this line if your cluster do not support it!)<br />
<code class="highlighter-rouge">#$ -q b.q</code><br />
<code class="highlighter-rouge">#$ -S /bin/bash</code><br />
<code class="highlighter-rouge">#$ -m abe</code><br />
<code class="highlighter-rouge">#$ -M &lt;e-mail&gt;</code><br />
<code class="highlighter-rouge">cd __MRO_JOB_WORKDIR__</code><br />
<code class="highlighter-rouge">source $HOME/cellranger-3.1.0/sourceme.bash</code></p>

<p>For clusters whose job managers do not support memory requests, it is possible to request memory 
in the form of cores via the <code class="highlighter-rouge">--mempercore</code> command-line option. This option scales up the number 
of threads requested via the <code class="highlighter-rouge">__MRO_THREADS__</code> variable according to how much memory a stage requires. 
see more at <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/cluster-mode">Cluster Mode</a></p>

<p><strong>4</strong>. Download single cell gene expression and reference genome datasets from <a href="https://www.10xgenomics.com/resources/datasets/">10XGenomics</a>.</p>

<p><strong>5</strong>. Create <code class="highlighter-rouge">sge.sh</code> file</p>

<p><code class="highlighter-rouge">TR="$HOME/refdata-cellranger-GRCh38-3.0.0"</code></p>

<p><strong>for Single Cell 3′ Gene expression</strong></p>

<p>Output files will appear in the out/ subdirectory within this pipeline output directory.</p>

<p><code class="highlighter-rouge">cd $HOME/10xgenomics/out</code></p>

<p>For pipeline output directory, the <code class="highlighter-rouge">--id</code> argument is used i.e 10XGTX_v3.</p>

<p><code class="highlighter-rouge">FASTQS="$HOME/pbmc_10k_v3_fastqs"</code></p>

<p><code class="highlighter-rouge">cellranger count --disable-ui \</code><br />
  <code class="highlighter-rouge">--id=10XGTX_v3 \</code><br />
  <code class="highlighter-rouge">--transcriptome=${TR} \</code><br />
  <code class="highlighter-rouge">--fastqs=${FASTQS} \</code><br />
  <code class="highlighter-rouge">--sample=pbmc_10k_v3 \</code><br />
  <code class="highlighter-rouge">--expect-cells=10000 \</code><br />
  <code class="highlighter-rouge">--jobmode=sge \</code><br />
  <code class="highlighter-rouge">--mempercore=8 \</code><br />
  <code class="highlighter-rouge">--jobinterval=5000 \</code><br />
  <code class="highlighter-rouge">--maxjobs=3</code></p>

<p><strong>for Single Cell 5′ gene expression</strong></p>

<p><code class="highlighter-rouge">FASTQS="$HOME/vdj_v1_hs_nsclc_5gex_fastqs"</code></p>

<h2 id="use-either-force-cells-or-expect-cells">use either –force-cells or –expect-cells</h2>

<p><code class="highlighter-rouge">cellranger count \</code><br />
  <code class="highlighter-rouge">--id=10XGTX_v5 \</code><br />
  <code class="highlighter-rouge">--fastqs=${FASTQS} \</code><br />
  <code class="highlighter-rouge">--transcriptome=${TR} \</code><br />
  <code class="highlighter-rouge">--sample=vdj_v1_hs_nsclc_5gex \</code><br />
  <code class="highlighter-rouge">--force-cells=7802 \</code><br />
  <code class="highlighter-rouge">--jobmode=sge \</code><br />
  <code class="highlighter-rouge">--mempercore=8 \</code><br />
  <code class="highlighter-rouge">--maxjobs=3 \</code><br />
  <code class="highlighter-rouge">--jobinterval=2000</code></p>

<p><strong>for Single Cell 5′ gene expression and cell surface protein (Feature Barcoding/Antibody Capture Assay)</strong></p>

<p><code class="highlighter-rouge">LIBRARY=$HOME/vdj_v1_hs_pbmc2_5gex_protein_fastqs/vdj_v1_hs_pbmc2_5gex_protein_library.csv</code><br />
<code class="highlighter-rouge">FEATURE_REF=$HOME/vdj_v1_hs_pbmc2_5gex_protein_fastqs/vdj_v1_hs_pbmc2_5gex_protein_feature_ref.csv</code></p>

<p><code class="highlighter-rouge">cellranger count \</code><br />
 <code class="highlighter-rouge">--libraries=${LIBRARY} \</code><br />
 <code class="highlighter-rouge">--feature-ref=${FEATURE_REF} \</code><br />
 <code class="highlighter-rouge">--id=PBMC_5GEX \</code><br />
 <code class="highlighter-rouge">--transcriptome=${TR} \</code><br />
 <code class="highlighter-rouge">--expect-cells=9000 \</code><br />
 <code class="highlighter-rouge">--jobmode=sge \</code><br />
 <code class="highlighter-rouge">--mempercore=8 \</code><br />
 <code class="highlighter-rouge">--maxjobs=3 \</code><br />
 <code class="highlighter-rouge">--jobinterval=5000</code></p>

<p><strong>6</strong>. Execute a command in screen and, detach and reconnect</p>

<p>Use <code class="highlighter-rouge">screen</code> command to get in/out of the system while keeping the processes running.</p>

<p><code class="highlighter-rouge">screen -S screen_name</code></p>

<p><code class="highlighter-rouge">bash sge.sh</code></p>

<p>If you want to exit the terminal without killing the running process, simply press <code class="highlighter-rouge">Ctrl+A+D</code>.</p>

<p>To reconnect to the screen: <code class="highlighter-rouge">screen -R screen_name</code></p>

<p><strong>7</strong>. Monitor work progress through a web browser</p>

<p>Open <code class="highlighter-rouge">_log</code> file present in output folder <code class="highlighter-rouge">PBMC_5GEX</code></p>

<p>If you see serving UI as <code class="highlighter-rouge">http://cluster.university.edu:3600?auth=rlSdT_QLzQ9O7fxEo-INTj1nQManinD21RzTAzkDVJ8</code>, then type the following from your laptop</p>

<p><code class="highlighter-rouge">ssh -NT -L 9000:cluster.university.edu:3600 user@cluster.university.edu</code></p>

<p><code class="highlighter-rouge">user@cluster.university.edu's password:</code></p>

<p>Then access the UI using the following URL in your web browser
<code class="highlighter-rouge">http://localhost:9000/</code></p>

<p><strong>8</strong>. Single Cell Integration in Seurat v3.0</p>

<p>Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data. Seurat aims to enable users to identify and interpret sources of heterogeneity from single cell transcriptomic measurements, and to integrate diverse types of single cell data. Seurat starts by reading cellranger data (barcodes.tsv.gz, features.tsv.gz and matrix.mtx.gz)</p>

<p><code class="highlighter-rouge">pbmc.data &lt;- Read10X(data.dir = "~/PBMC_5GEX/outs/filtered_feature_bc_matrix/")</code></p>

<hr />

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <p><a href="https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome">10XGenomics</a>&nbsp;<a href="#fnref:1" class="reversefootnote">&#8617;</a>&nbsp;<a href="#fnref:1:1" class="reversefootnote">&#8617;<sup>2</sup></a></p>
    </li>
    <li id="fn:2">
      <p><a href="https://satijalab.org/seurat/">Seurat</a>&nbsp;<a href="#fnref:2" class="reversefootnote">&#8617;</a></p>
    </li>
  </ol>
</div>

        
      </section>

      <footer class="page__meta">
        
        


  




  
  
  

  <p class="page__taxonomy">
    <strong><i class="fa fa-fw fa-tags" aria-hidden="true"></i> Tags: </strong>
    <span itemprop="keywords">
    
      
      
      <a href="http://localhost:4000/tags/#10xgenomics" class="page__taxonomy-item" rel="tag">10XGenomics</a><span class="sep">, </span>
    
      
      
      <a href="http://localhost:4000/tags/#cell-ranger" class="page__taxonomy-item" rel="tag">Cell Ranger</a><span class="sep">, </span>
    
      
      
      <a href="http://localhost:4000/tags/#chromium-single-cell-gene-expression" class="page__taxonomy-item" rel="tag">Chromium Single Cell Gene Expression</a><span class="sep">, </span>
    
      
      
      <a href="http://localhost:4000/tags/#cluster-computing" class="page__taxonomy-item" rel="tag">Cluster Computing</a><span class="sep">, </span>
    
      
      
      <a href="http://localhost:4000/tags/#feature-barcoding" class="page__taxonomy-item" rel="tag">Feature Barcoding</a><span class="sep">, </span>
    
      
      
      <a href="http://localhost:4000/tags/#seurat" class="page__taxonomy-item" rel="tag">Seurat</a><span class="sep">, </span>
    
      
      
      <a href="http://localhost:4000/tags/#single-cell-rna-sequencing-scrna-seq" class="page__taxonomy-item" rel="tag">Single cell RNA-sequencing (scRNA-seq)</a><span class="sep">, </span>
    
      
      
      <a href="http://localhost:4000/tags/#single-cell" class="page__taxonomy-item" rel="tag">Single Cell</a>
    
    </span>
  </p>




      </footer>

      

<section class="page__share">
  
    <h4 class="page__share-title">Share on</h4>
  

  <a href="https://twitter.com/intent/tweet?text=http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" class="btn btn--twitter" title="Share on Twitter"><i class="fa fa-fw fa-twitter" aria-hidden="true"></i><span> Twitter</span></a>

  <a href="https://www.facebook.com/sharer/sharer.php?u=http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" class="btn btn--facebook" title="Share on Facebook"><i class="fa fa-fw fa-facebook" aria-hidden="true"></i><span> Facebook</span></a>

  <a href="https://plus.google.com/share?url=http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" class="btn btn--google-plus" title="Share on Google Plus"><i class="fa fa-fw fa-google-plus" aria-hidden="true"></i><span> Google+</span></a>

  <a href="https://www.linkedin.com/shareArticle?mini=true&url=http://localhost:4000/posts/2019/01/blog-post-sc-ranseq/" class="btn btn--linkedin" title="Share on LinkedIn"><i class="fa fa-fw fa-linkedin" aria-hidden="true"></i><span> LinkedIn</span></a>
</section>

      


  <nav class="pagination">
    
      <a href="http://localhost:4000/posts/2019/01/blog-post-qiime2/" class="pagination--pager" title="Taxonomic and diversity profiling of the microbiome - 16S rRNA gene amplicon sequence data
">Previous</a>
    
    
      <a href="http://localhost:4000/posts/2019/12/blog-post-atacseq/" class="pagination--pager" title="ATAC-seq peak calling with MACS2
">Next</a>
    
  </nav>

    </div>

    
  </article>

  
  
</div>


    </script>

    <div class="page__footer">
      <footer>
        <!-- start custom footer snippets -->
<!--<a href="/sitemap/">Sitemap</a> -->
<!-- end custom footer snippets -->

        

<!--
<div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    
    
    
    
      <li><a href="http://github.com/adinasarapu"><i class="fa fa-fw fa-github" aria-hidden="true"></i> GitHub</a></li>
    
    
      <li><a href="http://bitbucket.org/adinasarapu"><i class="fa fa-fw fa-bitbucket" aria-hidden="true"></i> Bitbucket</a></li>
    
    <li><a href="http://localhost:4000/feed.xml"><i class="fa fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
  </ul>
</div>
-->

<div class="page__footer-copyright">&copy; 2020 Ashok R. Dinasarapu  | <a href="http://adinasarapu.github.io/terms/" rel="nofollow">Privacy Policy</a>

<!-- Powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://github.com/academicpages/academicpages.github.io">AcademicPages</a>, a fork of <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.--> 
</div>

      </footer>
    </div>

    <script src="http://localhost:4000/assets/js/main.min.js"></script>




  <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-68614074-1', 'auto');
  ga('send', 'pageview');
</script>






  </body>
</html>

