---
title: "Signatures of somatic mutations and gene expression from p16<sup>INK4A</sup> positive head and neck squamous cell carcinomas (HNSCC)."
collection: publications
permalink: /publication/2020-cancer
excerpt: 'Cancer Genomics'
date: 2020-8-20
tags:
  - Cancer
  - Head and Neck cancer
  - Genomics
  - RNA-seq
  - DNA-seq
  - bioinformatics
  - Emory Uiversity
venue: 'PLOS ONE'
paperurl: 'https://doi.org/10.1371/journal.pone.0238497'  
citation: 'Nabil F. Saba NF<sup>*</sup>, Dinasarapu AR<sup>*</sup>, Magliocca KR, Dwivedi B, Seby S, Qin ZS, Patel M, Griffith CC, Wang X, El-Deiry M, Steuer CE, Kowalski J, Shin DM, Zwick ME, Chen ZG (2020). &quot;Signatures of somatic mutations and gene expression from p16<sup>INK4A</sup> positive head and neck squamous cell carcinomas (HNSCC).&quot; <i>PLOS ONE</i> 15(9): e0238497.'  
---  

(*) These authors contributed equally to this work. 

Human papilloma virus (HPV) causes a subset of head and neck squamous cell carcinomas (HNSCC) of the oropharynx. We combined targeted DNA- and genome-wide RNA-sequencing to identify genetic variants and gene expression signatures respectively from patients with HNSCC including oropharyngeal squamous cell carcinomas (OPSCC). DNA and RNA were purified from 35- formalin fixed and paraffin embedded (FFPE) HNSCC tumor samples. Immuno-histochemical evaluation of tumors was performed to determine the expression levels of p16INK4A and classified tumor samples either p16+ or p16-. Using ClearSeq Comprehensive Cancer panel, we examined the distribution of somatic mutations. Somatic single-nucleotide variants (SNV) were called using GATK-Mutect2 (“tumor-only” mode) approach. Using RNA-seq, we identified a catalog of 1,044 and 8 genes as significantly expressed between p16+ and p16-, respectively at FDR 0.05 (5%) and 0.1 (10%). The clinicopathological characteristics of the patients including anatomical site, smoking and survival were analyzed when comparing p16+ and p16- tumors. The majority of tumors (65%) were p16+. Population sequence variant databases, including gnomAD, ExAC, COSMIC and dbSNP, were used to identify the mutational landscape of somatic sequence variants within sequenced genes. Hierarchical clustering of The Cancer Genome Atlas (TCGA) samples based on HPV-status was observed using differentially expressed genes. Using RNA-seq in parallel with targeted DNA-seq, we identified mutational and gene expression signatures characteristic of p16+ and p16- HNSCC. Our gene signatures are consistent with previously published data including TCGA and support the need to further explore the biologic relevance of these alterations in HNSCC.  

