

<!doctype html>
<html lang="en" class="no-js">
  <head>
    

<meta charset="utf-8">



<!-- begin SEO -->









<title>Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger) - Ashok R. Dinasarapu Ph.D</title>







<meta property="og:locale" content="en-US">
<meta property="og:site_name" content="Ashok R. Dinasarapu Ph.D">
<meta property="og:title" content="Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)">


  <link rel="canonical" href="https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/">
  <meta property="og:url" content="https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/">



  <meta property="og:description" content="Running spaceranger as cluster mode that uses Sun Grid Engine (SGE) as queuing.">





  

  





  <meta property="og:type" content="article">
  <meta property="article:published_time" content="2020-03-06T00:00:00-05:00">








  <script type="application/ld+json">
    {
      "@context" : "http://schema.org",
      "@type" : "Person",
      "name" : "Ashok R. Dinasarapu",
      "url" : "https://adinasarapu.github.io",
      "sameAs" : null
    }
  </script>






<!-- end SEO -->


<link href="https://adinasarapu.github.io/feed.xml" type="application/atom+xml" rel="alternate" title="Ashok R. Dinasarapu Ph.D Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="https://adinasarapu.github.io/assets/css/main.css">

<meta http-equiv="cleartype" content="on">
    

<!-- start custom head snippets -->

<link rel="apple-touch-icon" sizes="57x57" href="https://adinasarapu.github.io/images/apple-touch-icon-57x57.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="60x60" href="https://adinasarapu.github.io/images/apple-touch-icon-60x60.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="72x72" href="https://adinasarapu.github.io/images/apple-touch-icon-72x72.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="76x76" href="https://adinasarapu.github.io/images/apple-touch-icon-76x76.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="114x114" href="https://adinasarapu.github.io/images/apple-touch-icon-114x114.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="120x120" href="https://adinasarapu.github.io/images/apple-touch-icon-120x120.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="144x144" href="https://adinasarapu.github.io/images/apple-touch-icon-144x144.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="152x152" href="https://adinasarapu.github.io/images/apple-touch-icon-152x152.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="180x180" href="https://adinasarapu.github.io/images/apple-touch-icon-180x180.png?v=M44lzPylqQ">
<link rel="icon" type="image/png" href="https://adinasarapu.github.io/images/favicon-32x32.png?v=M44lzPylqQ" sizes="32x32">
<link rel="icon" type="image/png" href="https://adinasarapu.github.io/images/android-chrome-192x192.png?v=M44lzPylqQ" sizes="192x192">
<link rel="icon" type="image/png" href="https://adinasarapu.github.io/images/favicon-96x96.png?v=M44lzPylqQ" sizes="96x96">
<link rel="icon" type="image/png" href="https://adinasarapu.github.io/images/favicon-16x16.png?v=M44lzPylqQ" sizes="16x16">
<link rel="manifest" href="https://adinasarapu.github.io/images/manifest.json?v=M44lzPylqQ">
<link rel="mask-icon" href="https://adinasarapu.github.io/images/safari-pinned-tab.svg?v=M44lzPylqQ" color="#000000">
<link rel="shortcut icon" href="/images/favicon.ico?v=M44lzPylqQ">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-TileImage" content="https://adinasarapu.github.io/images/mstile-144x144.png?v=M44lzPylqQ">
<meta name="msapplication-config" content="https://adinasarapu.github.io/images/browserconfig.xml?v=M44lzPylqQ">
<meta name="theme-color" content="#ffffff">
<link rel="stylesheet" href="https://adinasarapu.github.io/assets/css/academicons.css"/>

<script type="text/x-mathjax-config"> MathJax.Hub.Config({ TeX: { equationNumbers: { autoNumber: "all" } } }); </script>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/latest.js?config=TeX-MML-AM_CHTML' async></script>

<!-- end custom head snippets -->

  </head>

  <body>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->
    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <button><div class="navicon"></div></button>
        <ul class="visible-links">
          <li class="masthead__menu-item masthead__menu-item--lg"><a href="https://adinasarapu.github.io/">Ashok R. Dinasarapu Ph.D</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/publications/">Publications</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/teaching/">Teaching</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/portfolio/">Big Data</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/year-archive/">NGS | Proteomics</a></li>
          
            
            <li class="masthead__menu-item"><a href="https://adinasarapu.github.io/cv/">Résumé</a></li>
          
        </ul>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>

    





<div id="main" role="main">
  


  <div class="sidebar sticky">
  



<div itemscope itemtype="http://schema.org/Person">

  <div class="author__avatar">
    
    	<img src="https://adinasarapu.github.io/images/profile.png" class="author__avatar" alt="Scientist, Bioinformatics">
    
  </div>

  <div class="author__content">
    <h3 class="author__name">Scientist, Bioinformatics</h3>
    <p class="author__bio">Bridging the gap between Genomics and Data Science</p>
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      
        <li><i class="fa fa-fw fa-map-marker" aria-hidden="true"></i> Atlanta, GA</li>
      
      
      
      
        <li><a href="mailto:ashok.reddy.dinasarapu@emory.edu"><i class="fas fa-fw fa-envelope" aria-hidden="true"></i> Email</a></li>
      
      
       
        <li><a href="https://www.researchgate.net/profile/Ashok_Dinasarapu"><i class="fab fa-fw fa-researchgate" aria-hidden="true"></i> ResearchGate</a></li>
      
      
        <li><a href="https://twitter.com/adinasarapu"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
      
      
      
      
        <li><a href="https://www.linkedin.com/in/dareddy"><i class="fab fa-fw fa-linkedin" aria-hidden="true"></i> LinkedIn</a></li>
      
      
      
      
      
        <li><a href="https://bitbucket.org/adinasarapu"><i class="fab fa-fw fa-bitbucket" aria-hidden="true"></i> Bitbucket</a></li>
      
      
        <li><a href="https://github.com/adinasarapu"><i class="fab fa-fw fa-github" aria-hidden="true"></i> Github</a></li>
      
      
      
      
      
      
      
      
      
      
      
      
      
      
        <li><a href="https://scholar.google.com/citations?user=b6GBykAAAAAJ"><i class="fas fa-fw fa-graduation-cap"></i> Google Scholar</a></li>
      
      
        <li><a href="https://pubmed.ncbi.nlm.nih.gov/?term=dinasarapu+ashok+OR+D+ashok+reddy&sort=pubdate"><i class="ai ai-pubmed-square ai-fw"></i> PubMed</a></li>
      
      
        <li><a href="http://orcid.org/0000-0002-1423-1518"><i class="ai ai-orcid-square ai-fw"></i> ORCID</a></li>
      
      
      
    </ul>
  </div>
</div>

  
  </div>


  <article class="page" itemscope itemtype="http://schema.org/CreativeWork">
    <meta itemprop="headline" content="Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)">
    <meta itemprop="description" content="Running spaceranger as cluster mode that uses Sun Grid Engine (SGE) as queuing.">
    <meta itemprop="datePublished" content="March 06, 2020">
    

    <div class="page__inner-wrap">
      
        <header>
          <h1 class="page__title" itemprop="headline">Spatial gene expression data analysis on Cluster (10X Genomics, Space Ranger)
</h1>
          
            <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
          
        
        
        
          <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-03-06T00:00:00-05:00">March 06, 2020</time></p>
        
        
             
        
    
        </header>
      

      <section class="page__content" itemprop="text">
        <p>Running <a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">spaceranger</a> as cluster mode that uses Sun Grid Engine (SGE) as queuing.</p>

<p>There are 2 steps to analyze Spatial RNA-seq data<sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup>.</p>

<p><strong>Step 1</strong>: <code class="language-plaintext highlighter-rouge">spaceranger mkfastq</code> demultiplexes raw base call (<code class="language-plaintext highlighter-rouge">BCL</code>) files generated by Illumina sequencers into FASTQ files.<br />
<strong>Step 2</strong>: <code class="language-plaintext highlighter-rouge">spaceranger count</code> takes FASTQ files from <code class="language-plaintext highlighter-rouge">spaceranger mkfastq</code> and performs alignment, filtering, barcode counting, and UMI counting.</p>

<p>Running pipelines on cluster requires the following:</p>

<p><strong>1</strong>. Load Space Ranger module (<code class="language-plaintext highlighter-rouge">spaceranger-1.0.0</code>)<sup id="fnref:1:1"><a href="#fn:1" class="footnote">1</a></sup> or, download and uncompress spaceranger at your <code class="language-plaintext highlighter-rouge">$HOME</code> directory and add PATH in <code class="language-plaintext highlighter-rouge">~/.bashrc</code>.</p>

<p><strong>2</strong>. Update job config file (<code class="language-plaintext highlighter-rouge">spaceranger-1.0.0/external/martian/jobmanagers/config.json</code>) for threads and memory. For example</p>

<p><code class="language-plaintext highlighter-rouge">"threads_per_job": 8,</code><br />
<code class="language-plaintext highlighter-rouge">"memGB_per_job": 64,</code></p>

<p><strong>3</strong>. Update template file (<code class="language-plaintext highlighter-rouge">spaceranger-1.0.0/external/martian/jobmanagers/sge.template</code>).</p>

<p><code class="language-plaintext highlighter-rouge">#!/bin/bash</code><br />
<code class="language-plaintext highlighter-rouge">#$ -pe smp __MRO_THREADS__</code><br />
<code class="language-plaintext highlighter-rouge">##$ -l mem_free=__MRO_MEM_GB__G</code> (comment this line if your cluster do not support it!)<br />
<code class="language-plaintext highlighter-rouge">#$ -q b.q</code><br />
<code class="language-plaintext highlighter-rouge">#$ -S /bin/bash</code><br />
<code class="language-plaintext highlighter-rouge">#$ -m abe</code><br />
<code class="language-plaintext highlighter-rouge">#$ -M &lt;e-mail&gt;</code><br />
<code class="language-plaintext highlighter-rouge">cd __MRO_JOB_WORKDIR__</code><br />
<code class="language-plaintext highlighter-rouge">source ../spaceranger-1.0.0/sourceme.bash</code> (update with complete path)</p>

<p><em>For clusters whose job managers do not support memory requests, it is possible to request memory 
in the form of cores via the <code class="language-plaintext highlighter-rouge">--mempercore</code> command-line option. This option scales up the number 
of threads requested via the <code class="language-plaintext highlighter-rouge">__MRO_THREADS__</code> variable according to how much memory a stage requires</em>.<br />
Read more at <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/cluster-mode">Cluster Mode</a></p>

<p><strong>4</strong>. Download spatial gene expression, image file and reference genome datasets from <a href="https://www.10xgenomics.com/resources/datasets/">10XGenomics</a>.</p>

<p><strong>5</strong>. Create <code class="language-plaintext highlighter-rouge">sge.sh</code> file</p>

<p><code class="language-plaintext highlighter-rouge">TR="$HOME/refdata-cellranger-mm10-3.0.0"</code></p>

<p>Output files will appear in the out/ subdirectory within this pipeline output directory.</p>

<p><code class="language-plaintext highlighter-rouge">cd $HOME/10xgenomics/out</code></p>

<p>For pipeline output directory, the <code class="language-plaintext highlighter-rouge">--id</code> argument is used i.e Adult_Mouse_Brain.</p>

<p><code class="language-plaintext highlighter-rouge">FASTQS="$HOME/V1_Adult_Mouse_Brain_fastqs"</code></p>

<p><code class="language-plaintext highlighter-rouge">spaceranger count --disable-ui \</code><br />
  <code class="language-plaintext highlighter-rouge">--id=Adult_Mouse_Brain \</code><br />
  <code class="language-plaintext highlighter-rouge">--transcriptome=${TR} \</code><br />
  <code class="language-plaintext highlighter-rouge">--fastqs=${FASTQS} \</code><br />
  <code class="language-plaintext highlighter-rouge">--sample=V1_Adult_Mouse_Brain \</code><br />
  <code class="language-plaintext highlighter-rouge">--image=$DATA_DIR/V1_Adult_Mouse_Brain_image.tif \</code><br />
  <code class="language-plaintext highlighter-rouge">--slide=V19L01-041 \</code><br />
  <code class="language-plaintext highlighter-rouge">--area=C1 \</code><br />
  <code class="language-plaintext highlighter-rouge">--jobmode=sge \</code><br />
  <code class="language-plaintext highlighter-rouge">--mempercore=8 \</code><br />
  <code class="language-plaintext highlighter-rouge">--jobinterval=5000 \</code><br />
  <code class="language-plaintext highlighter-rouge">--maxjobs=3</code></p>

<p><strong>6</strong>. Execute a command in screen and, detach and reconnect</p>

<p>Use <code class="language-plaintext highlighter-rouge">screen</code> command to get in/out of the system while keeping the processes running.</p>

<p><code class="language-plaintext highlighter-rouge">screen -S screen_name</code></p>

<p><code class="language-plaintext highlighter-rouge">bash sge.sh</code></p>

<p>If you want to exit the terminal without killing the running process, simply press <code class="language-plaintext highlighter-rouge">Ctrl+A+D</code>.</p>

<p>To reconnect to the screen: <code class="language-plaintext highlighter-rouge">screen -R screen_name</code></p>

<p><strong>7</strong>. Monitor work progress through a web browser</p>

<p>Open <code class="language-plaintext highlighter-rouge">_log</code> file present in output folder <code class="language-plaintext highlighter-rouge">Adult_Mouse_Brain</code></p>

<p>If you see serving UI as <code class="language-plaintext highlighter-rouge">http://cluster.university.edu:3600?auth=rlSdT_QLzQ9O7fxEo-INTj1nQManinD21RzTAzkDVJ8</code>, then type the following from your laptop</p>

<p><code class="language-plaintext highlighter-rouge">ssh -NT -L 9000:cluster.university.edu:3600 user@cluster.university.edu</code></p>

<p><code class="language-plaintext highlighter-rouge">user@cluster.university.edu's password:</code></p>

<p>Then access the UI using the following URL in your web browser
<code class="language-plaintext highlighter-rouge">http://localhost:9000/</code></p>

<hr />

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <p><a href="https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/what-is-space-ranger">10XGenomics- Visium spatial RNA-seq</a> <a href="#fnref:1" class="reversefootnote">&#8617;</a> <a href="#fnref:1:1" class="reversefootnote">&#8617;<sup>2</sup></a></p>
    </li>
  </ol>
</div>

        
      </section>

      <footer class="page__meta">
        
        


  




  
  
  

  <p class="page__taxonomy">
    <strong><i class="fa fa-fw fa-tags" aria-hidden="true"></i> Tags: </strong>
    <span itemprop="keywords">
    
      
      
      <a href="https://adinasarapu.github.io/tags/#10xgenomics" class="page__taxonomy-item" rel="tag">10XGenomics</a><span class="sep">, </span>
    
      
      
      <a href="https://adinasarapu.github.io/tags/#brightfield-microscope-image" class="page__taxonomy-item" rel="tag">brightfield microscope image</a><span class="sep">, </span>
    
      
      
      <a href="https://adinasarapu.github.io/tags/#cluster-computing" class="page__taxonomy-item" rel="tag">Cluster Computing</a><span class="sep">, </span>
    
      
      
      <a href="https://adinasarapu.github.io/tags/#sge-cluster" class="page__taxonomy-item" rel="tag">SGE Cluster</a><span class="sep">, </span>
    
      
      
      <a href="https://adinasarapu.github.io/tags/#space-ranger" class="page__taxonomy-item" rel="tag">Space Ranger</a><span class="sep">, </span>
    
      
      
      <a href="https://adinasarapu.github.io/tags/#spatial-gene-expression" class="page__taxonomy-item" rel="tag">SPATIAL GENE EXPRESSION</a><span class="sep">, </span>
    
      
      
      <a href="https://adinasarapu.github.io/tags/#visium-spatial-gene-expression-solution" class="page__taxonomy-item" rel="tag">Visium Spatial Gene Expression Solution</a>
    
    </span>
  </p>




      </footer>

      

<section class="page__share">
  
    <h4 class="page__share-title">Share on</h4>
  

  <a href="https://twitter.com/intent/tweet?text=https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" class="btn btn--twitter" title="Share on Twitter"><i class="fab fa-twitter" aria-hidden="true"></i><span> Twitter</span></a>

  <a href="https://www.facebook.com/sharer/sharer.php?u=https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" class="btn btn--facebook" title="Share on Facebook"><i class="fab fa-facebook" aria-hidden="true"></i><span> Facebook</span></a>

  <a href="https://www.linkedin.com/shareArticle?mini=true&url=https://adinasarapu.github.io/posts/2020/03/blog-spatial-gene-expression/" class="btn btn--linkedin" title="Share on LinkedIn"><i class="fab fa-linkedin" aria-hidden="true"></i><span> LinkedIn</span></a>
</section>

      


  <nav class="pagination">
    
      <a href="https://adinasarapu.github.io/posts/2019/12/blog-post-atacseq/" class="pagination--pager" title="ATAC-seq peak calling with MACS2
">Previous</a>
    
    
      <a href="https://adinasarapu.github.io/posts/2018/04/blog-post-lfq/" class="pagination--pager" title="Quantitative proteomics: label-free quantitation of proteins
">Next</a>
    
  </nav>

    </div>

    
  </article>

  
  
    <div class="page__related">
      
        <h4 class="page__related-title">You May Also Enjoy</h4>
      
      <div class="grid__wrapper">
        
          





<div class="grid__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2023/02/blog-post-somalogic-proteomics/" rel="permalink">Quantitative proteomics: aptamer-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  6 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2023-02-20T00:00:00-05:00">February 20, 2023</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>The aptamer-based <a href="https://somalogic.com">SomaScan®</a> assay is one of the popular methods of measuring abundances of protein targets. There is very little information on correlation between mass spectrometry (MS)-based proteomics, SOMAscan and Olink assays; Olink is another popular high throughput antibody-based platform. Some studies also reported a measurement variation between those platforms. In general aptamers/SOMAmers are selected against target proteins in their native conformation and in some cases against a functional protein with “known” post translational modifications (PTMs). It’s well known that novel PTMs (pathogen or disease-induced) can impact the protein structure, electrophilicity and interactions with proteins. The other main disadvantage is quantification which is based on DNA microarray chips (background noise). The main advantages are lower cost and data analysis.</p>

</p>
    
    
    

  </article>
</div>

        
          





<div class="grid__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2022/10/blog-post-kaplan-meier-curve/" rel="permalink">Kaplan-Meier Curve using R
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  2 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2022-10-19T00:00:00-04:00">October 19, 2022</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Kaplan-Meier curve shows what the probability of an event (for example, survival) at a certain time interval. The log-rank test compares the survival curves of two or more groups. With a small subset of patients, the Kaplan-Meier estimates can be misleading and should be interpreted with caution.</p>

</p>
    
    
    

  </article>
</div>

        
          





<div class="grid__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2021/04/blog-post-vep/" rel="permalink">Annotation of genetic variants
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2021-04-13T00:00:00-04:00">April 13, 2021</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Tools such as ANNOVAR, Variant Effect Predictor (VEP) or SnpEff annotate genetic variants (SNPs, INDELS, CNVs etc) present in VCF file. These tools integrate the annotations within the INFO column of the original VCF file.</p>

</p>
    
    
    

  </article>
</div>

        
          





<div class="grid__item">
  <article class="archive__item" itemscope itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="https://adinasarapu.github.io/posts/2020/01/blog-post-tmt/" rel="permalink">Quantitative proteomics: TMT-based quantitation of proteins
</a>
      
    </h2>
    
    
      <p class="page__meta"><i class="fa fa-clock-o" aria-hidden="true"></i> 


  
	  4 minute read
	
</p>
    

        
         <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2020-04-04T00:00:00-04:00">April 04, 2020</time></p>
        

    
    <p class="archive__item-excerpt" itemprop="description"><p>Quantification of proteins using isobaric labeling (tandem mass tag or TMT) starts with the reduction of disulfide bonds in proteins with Dithiothreitol (DTT).</p>

</p>
    
    
    

  </article>
</div>

        
      </div>
    </div>
  
</div>


    <div class="page__footer">
      <footer>
        <!-- start custom footer snippets -->
<!--<a href="/cv/">Resume</a> -->
<!-- end custom footer snippets -->

        

<div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    
    
    
    
      <li><a href="http://github.com/adinasarapu"><i class="fab fa-github" aria-hidden="true"></i> GitHub</a></li>
    
    
      <li><a href="http://bitbucket.org/adinasarapu"><i class="fab fa-bitbucket" aria-hidden="true"></i> Bitbucket</a></li>
    
    <!--<li><a href="https://adinasarapu.github.io/feed.xml">
	<i class="fa fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>-->
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2023 Ashok R. Dinasarapu. 101 Woodruff Circle, Woodruff Memorial Research Building,  Room#6302, Emory University, Atlanta, GA 30322.</div>

      </footer>
    </div>

    <script src="https://adinasarapu.github.io/assets/js/main.min.js"></script>




  <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'G-RWM39QLMPF', 'auto');
  ga('send', 'pageview');
</script>






  </body>
</html>

